<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>2023.knit</title>

<script src="site_libs/header-attrs-2.13/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Research Progress</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Progress
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="2022.html">2022</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/wflowr_template">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-06-26
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>wflowr_template/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220610code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20220610)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220610code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220610)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliangendTransWIPtreecfe9563a1eb35f911357cb06b3460fa1e8551723targetblankcfe9563a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/liangend/Trans_WIP/tree/cfe9563a1eb35f911357cb06b3460fa1e8551723" target="_blank">cfe9563</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomliangendTransWIPtreecfe9563a1eb35f911357cb06b3460fa1e8551723targetblankcfe9563a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/liangend/Trans_WIP/tree/cfe9563a1eb35f911357cb06b3460fa1e8551723" target="_blank">cfe9563</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
working directory clean
</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/2023.Rmd</code>) and HTML (<code>docs/2023.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/cfe9563a1eb35f911357cb06b3460fa1e8551723/analysis/2023.Rmd" target="_blank">cfe9563</a>
</td>
<td>
liangend
</td>
<td>
2023-06-26
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/33e35206689f3ab58f5879921f93ecdf69c48215/docs/2023.html" target="_blank">33e3520</a>
</td>
<td>
liangend
</td>
<td>
2023-06-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/66419b724f3ec01a318770f102339091342a094f/analysis/2023.Rmd" target="_blank">66419b7</a>
</td>
<td>
liangend
</td>
<td>
2023-06-21
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/f59ac017cbb565f04bc830d2116b347f0792ae0b/docs/2023.html" target="_blank">f59ac01</a>
</td>
<td>
liangend
</td>
<td>
2023-06-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/6dad5246ba79e92f515cbcb08a7c31950b149556/analysis/2023.Rmd" target="_blank">6dad524</a>
</td>
<td>
liangend
</td>
<td>
2023-06-12
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/cef86d52aad8e5bd99699f963f164a39cd1f59f7/docs/2023.html" target="_blank">cef86d5</a>
</td>
<td>
liangend
</td>
<td>
2023-06-09
</td>
<td>
commit message
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/09f9f32e561939bb951712048d9d678d97b38bac/analysis/2023.Rmd" target="_blank">09f9f32</a>
</td>
<td>
liangend
</td>
<td>
2023-06-09
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/2494cdb941f70abe1cbd906befe0ee51c4458d11/docs/2023.html" target="_blank">2494cdb</a>
</td>
<td>
liangend
</td>
<td>
2023-05-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/e2a33e9d8cccd23f105be00a60b243f53b0ef575/analysis/2023.Rmd" target="_blank">e2a33e9</a>
</td>
<td>
liangend
</td>
<td>
2023-05-29
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/039514872bfb7de0b73f9354d51b834f00de49db/docs/2023.html" target="_blank">0395148</a>
</td>
<td>
liangend
</td>
<td>
2023-05-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/ec5046f3b412284994b03b08cfe7aa988be0841a/analysis/2023.Rmd" target="_blank">ec5046f</a>
</td>
<td>
liangend
</td>
<td>
2023-05-23
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/7c0e1f7afd5388b047a0258d253bd612a9f648a2/docs/2023.html" target="_blank">7c0e1f7</a>
</td>
<td>
liangend
</td>
<td>
2023-05-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/656941dc293de82a818f3020efce949b0828c3d6/analysis/2023.Rmd" target="_blank">656941d</a>
</td>
<td>
liangend
</td>
<td>
2023-05-17
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/2f7dac835eb803c2311321b0fa1d2e2fa3f9e70e/docs/2023.html" target="_blank">2f7dac8</a>
</td>
<td>
liangend
</td>
<td>
2023-05-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/f88dbdcea6cfbe56f22af46cb7e95945c72d170b/docs/2023.html" target="_blank">f88dbdc</a>
</td>
<td>
liangend
</td>
<td>
2023-05-11
</td>
<td>
commit message
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/d0601b58bfa0442676d7f5e9bf6b9d5517f12020/analysis/2023.Rmd" target="_blank">d0601b5</a>
</td>
<td>
liangend
</td>
<td>
2023-05-11
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/8b19366b4dd7fdaa63ad2cc6504cc145d633ada4/docs/2023.html" target="_blank">8b19366</a>
</td>
<td>
liangend
</td>
<td>
2023-05-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/4609330edd1613f67c665fa4db67f4218e7f19b7/analysis/2023.Rmd" target="_blank">4609330</a>
</td>
<td>
liangend
</td>
<td>
2023-05-09
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/712224ea364df0c85eb839dc437300b0cb9d47e4/docs/2023.html" target="_blank">712224e</a>
</td>
<td>
liangend
</td>
<td>
2023-05-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/ccd223b2ed4ccac8f111e0e437ed5e0f853f9da4/analysis/2023.Rmd" target="_blank">ccd223b</a>
</td>
<td>
liangend
</td>
<td>
2023-05-04
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/a7d750caf4debff9a2886d9a5ba91882c79e5043/docs/2023.html" target="_blank">a7d750c</a>
</td>
<td>
liangend
</td>
<td>
2023-05-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/b8cd27cb07d2d262539f246f1a5b2f06a671dbd3/analysis/2023.Rmd" target="_blank">b8cd27c</a>
</td>
<td>
liangend
</td>
<td>
2023-05-04
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/27c8af89e4b21f5146ef63d00ed2c31b620eeb3a/docs/2023.html" target="_blank">27c8af8</a>
</td>
<td>
liangend
</td>
<td>
2023-05-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/8c9fefdf3c171b1b58a40020129449bd8e14438a/docs/2023.html" target="_blank">8c9fefd</a>
</td>
<td>
liangend
</td>
<td>
2023-05-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/2b1451f9efc5319ca3e8a0c9469035b8a4b345e4/analysis/2023.Rmd" target="_blank">2b1451f</a>
</td>
<td>
liangend
</td>
<td>
2023-05-04
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/a42b6fbe95add507175fb4a51d983615f8cc973d/docs/2023.html" target="_blank">a42b6fb</a>
</td>
<td>
liangend
</td>
<td>
2023-04-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/eca4fc1a2d8bf8c98f2040fff35c2e33dfc9beba/analysis/2023.Rmd" target="_blank">eca4fc1</a>
</td>
<td>
liangend
</td>
<td>
2023-04-27
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/77cb528a40256a4d7878b77149dffa76d40e9b84/docs/2023.html" target="_blank">77cb528</a>
</td>
<td>
liangend
</td>
<td>
2023-04-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/225575db8a3c1b6450dc31628b9aa4af8f7562cc/analysis/2023.Rmd" target="_blank">225575d</a>
</td>
<td>
liangend
</td>
<td>
2023-04-27
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/309938a8b90dcd7f04a3472f5590587be4b81871/docs/2023.html" target="_blank">309938a</a>
</td>
<td>
liangend
</td>
<td>
2023-04-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/fc3b7d329231f70edf783a3eeca37e20adfd53b9/analysis/2023.Rmd" target="_blank">fc3b7d3</a>
</td>
<td>
liangend
</td>
<td>
2023-04-20
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/f22c4a7610f94d3fa895bf2c643f5896dfe3f2e6/docs/2023.html" target="_blank">f22c4a7</a>
</td>
<td>
liangend
</td>
<td>
2023-04-20
</td>
<td>
commit message
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/368102f06372f249a0fc343efd3a91c5bda748cf/analysis/2023.Rmd" target="_blank">368102f</a>
</td>
<td>
liangend
</td>
<td>
2023-04-20
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/30a8138be88c1fa64a59e29a2ec20c8835ae41af/docs/2023.html" target="_blank">30a8138</a>
</td>
<td>
liangend
</td>
<td>
2023-04-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/b154091037f821092d809d913554ca4a2f7db8a8/analysis/2023.Rmd" target="_blank">b154091</a>
</td>
<td>
liangend
</td>
<td>
2023-04-12
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/c94e9f8620d467ff350d330beb962b067083cc72/docs/2023.html" target="_blank">c94e9f8</a>
</td>
<td>
liangend
</td>
<td>
2023-04-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/62167b1297c489e67abd0abc74bc47cc542375b7/analysis/2023.Rmd" target="_blank">62167b1</a>
</td>
<td>
liangend
</td>
<td>
2023-04-10
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/b1ee857eb724676c1a51e525c8964460bff8a8f5/docs/2023.html" target="_blank">b1ee857</a>
</td>
<td>
liangend
</td>
<td>
2023-04-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/771f5184d245f47c03c183994568d783e226eb3b/analysis/2023.Rmd" target="_blank">771f518</a>
</td>
<td>
liangend
</td>
<td>
2023-04-05
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/7509421f9653ac7f86a6c53464eee54e3582f433/docs/2023.html" target="_blank">7509421</a>
</td>
<td>
liangend
</td>
<td>
2023-03-31
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/3fe49c44b79690cefc06637a399a247f97f42f4d/analysis/2023.Rmd" target="_blank">3fe49c4</a>
</td>
<td>
liangend
</td>
<td>
2023-03-31
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/f3756fb3620da451bd113264d091a443a2e26572/docs/2023.html" target="_blank">f3756fb</a>
</td>
<td>
liangend
</td>
<td>
2023-03-31
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/7492d1ff19f70d80dc64d96c708ca1e2df4a9503/analysis/2023.Rmd" target="_blank">7492d1f</a>
</td>
<td>
liangend
</td>
<td>
2023-03-31
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/8c3ea4faf833d1eddd06c7efa8629b2d4daf6538/docs/2023.html" target="_blank">8c3ea4f</a>
</td>
<td>
liangend
</td>
<td>
2023-03-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/39d5d49d9ececf2a1d54165d43c9f4595f2ea3cf/analysis/2023.Rmd" target="_blank">39d5d49</a>
</td>
<td>
liangend
</td>
<td>
2023-03-24
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/0c83f1d048fe5a193bfc1579c80cc9e7458704ac/docs/2023.html" target="_blank">0c83f1d</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/27c2a173aa23696c9350e1cb0de261589d21a202/analysis/2023.Rmd" target="_blank">27c2a17</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/9ffc5c1b5fc2cab9b9d04c4acd82c1237e834460/docs/2023.html" target="_blank">9ffc5c1</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/323c5c0fbb78a84ad3778b8bec7460b210d9cab8/analysis/2023.Rmd" target="_blank">323c5c0</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/afdcb8e50517efabdce69f8f7eca0b2fcf43ae84/docs/2023.html" target="_blank">afdcb8e</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/b5effeb22b45065d5b1186c26654c10579fa9d0d/analysis/2023.Rmd" target="_blank">b5effeb</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/1dec8cb68498836bec5b8baceee5f1ee2c2fd290/docs/2023.html" target="_blank">1dec8cb</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/d2e487c6bf0c2b64e2b61bc2e1658009b1467e04/analysis/2023.Rmd" target="_blank">d2e487c</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/9e893ed2d35bd4aa49c1da8c8a3482e0efbc1df3/docs/2023.html" target="_blank">9e893ed</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/cf828cf9d0655f30dcfcbd32c7748467f18f13ea/analysis/2023.Rmd" target="_blank">cf828cf</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/0f711dd1326614d4e911c1004f4ac66c28c292a0/docs/2023.html" target="_blank">0f711dd</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/24f4d30d79828c178e1ee8b9c971697f732e64e0/analysis/2023.Rmd" target="_blank">24f4d30</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/fd02842eda684b3682830dc287802023f7cd30af/docs/2023.html" target="_blank">fd02842</a>
</td>
<td>
liangend
</td>
<td>
2023-03-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/5f2d0e726fe0988b30db02a9bf00aee31d4642cd/analysis/2023.Rmd" target="_blank">5f2d0e7</a>
</td>
<td>
liangend
</td>
<td>
2023-03-15
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/2d25571b1d32700cad2db4c43328c614fd2a26e0/docs/2023.html" target="_blank">2d25571</a>
</td>
<td>
liangend
</td>
<td>
2023-03-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/7198058432857fad10201591f8ed6e576d6a5f75/analysis/2023.Rmd" target="_blank">7198058</a>
</td>
<td>
liangend
</td>
<td>
2023-03-15
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/1775697031a861ab2735943f3f0fc74be821f5ca/docs/2023.html" target="_blank">1775697</a>
</td>
<td>
liangend
</td>
<td>
2023-03-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/d2050092cc1f9b74b94eaf7d22f85305faf595f4/analysis/2023.Rmd" target="_blank">d205009</a>
</td>
<td>
liangend
</td>
<td>
2023-03-12
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/7a8dbdbdc586f3e1e8ddf50c81381d92e0707a12/docs/2023.html" target="_blank">7a8dbdb</a>
</td>
<td>
liangend
</td>
<td>
2023-03-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/e404203ec19cb2778ff3f25ce8cd5340fc4667a2/analysis/2023.Rmd" target="_blank">e404203</a>
</td>
<td>
liangend
</td>
<td>
2023-03-03
</td>
<td>
update
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/da48d1deacc7276d19f042d760b2c0c8afce47ce/analysis/2023.Rmd" target="_blank">da48d1d</a>
</td>
<td>
liangend
</td>
<td>
2023-03-03
</td>
<td>
commit message
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/e2d739b2d8bcb657829b77d12b2f2526d4d91055/docs/2023.html" target="_blank">e2d739b</a>
</td>
<td>
liangend
</td>
<td>
2023-03-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/c3dac56b6f16d40158ad0b4416eb1381b14b7934/analysis/2023.Rmd" target="_blank">c3dac56</a>
</td>
<td>
liangend
</td>
<td>
2023-03-03
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/16578009854aa44b696efd44f63bb7dcfdaede5d/docs/2023.html" target="_blank">1657800</a>
</td>
<td>
liangend
</td>
<td>
2023-02-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/d7445f90487b72fbb13436e3972214c67e764ad0/analysis/2023.Rmd" target="_blank">d7445f9</a>
</td>
<td>
liangend
</td>
<td>
2023-02-23
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/925729dcd6caff2ab502cbcc9fd014c7b2ea24be/docs/2023.html" target="_blank">925729d</a>
</td>
<td>
liangend
</td>
<td>
2023-02-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/b1e9c83d3b33453c840f1b16c3b5c78fe3d18646/analysis/2023.Rmd" target="_blank">b1e9c83</a>
</td>
<td>
liangend
</td>
<td>
2023-02-23
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/f510f95bc576bb9dc464bcecc212520be681ae54/docs/2023.html" target="_blank">f510f95</a>
</td>
<td>
liangend
</td>
<td>
2023-02-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/a7b920cdbc123cedc06e8698944fac2bb3b68210/analysis/2023.Rmd" target="_blank">a7b920c</a>
</td>
<td>
liangend
</td>
<td>
2023-02-22
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/15a5da6327bcb16b55ed22272fd76c0bf1af8381/docs/2023.html" target="_blank">15a5da6</a>
</td>
<td>
liangend
</td>
<td>
2023-02-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/8e649daed1401495c861623b9aa71261a6a88a28/docs/2023.html" target="_blank">8e649da</a>
</td>
<td>
liangend
</td>
<td>
2023-02-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/5dff0f73ab8c5b28909cf4ef931a88b64708c7f8/docs/2023.html" target="_blank">5dff0f7</a>
</td>
<td>
liangend
</td>
<td>
2023-02-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/cb8bb8bd6da777b7f1f95abded2ca220904749d7/analysis/2023.Rmd" target="_blank">cb8bb8b</a>
</td>
<td>
liangend
</td>
<td>
2023-02-21
</td>
<td>
update
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
</style>
<div id="section" class="section level2">
<h2>6/21/2023</h2>
<div id="analyzing-gtex-using-eur-only" class="section level3">
<h3>1. Analyzing GTEx using EUR only</h3>
Fewer signals were found when decreasing the sample size from all (702) to EUR only (599) for muscle. Similarly for blood (670 vs 574).
<div class="figure" style="text-align: center">
<img src="assets/eur_vs_all_pco.png" alt="Number of significant modules and pairs using all/EUR individuals (16_all_vs_eur_ind.R)"  />
<p class="caption">
Number of significant modules and pairs using all/EUR individuals (16_all_vs_eur_ind.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-eur_vs_all_pco">
Past versions of eur_vs_all_pco.png
</button>
</p>
<div id="fig-eur_vs_all_pco" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/e77de6d1b68afef38b7ee1bad3c36419a713bf4d/docs/assets/eur_vs_all_pco.png" target="_blank">e77de6d</a>
</td>
<td>
liangend
</td>
<td>
2023-06-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
For DGN, I randomly subset 574 individuals (equal to GTEx blood EUR sample size), and compared the univariate results. Among all the common genes, DGN dataset had more small p values. For number of eGenes, DGN had 177 and GTEx had 23 (fdr &lt; 0.1).
<div class="figure" style="text-align: center">
<img src="assets/gtex_dgn_univar.png" alt="Number of significant SNP-gene pairs using the same sample size for GTEx blood and DGN (16_all_vs_eur_ind.R)"  />
<p class="caption">
Number of significant SNP-gene pairs using the same sample size for GTEx blood and DGN (16_all_vs_eur_ind.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-gtex_dgn_univar">
Past versions of gtex_dgn_univar.png
</button>
</p>
<div id="fig-gtex_dgn_univar" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/e77de6d1b68afef38b7ee1bad3c36419a713bf4d/docs/assets/gtex_dgn_univar.png" target="_blank">e77de6d</a>
</td>
<td>
liangend
</td>
<td>
2023-06-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="geuvadis-dataset-analysis" class="section level3">
<h3>2. Geuvadis dataset analysis</h3>
<p>Geuvadis dataset contains 358 individuals and 12,382 genes for analysis. The gene expression count was not corrected for mappability. Out of 211 gene modules, there are 159 significant eModules and 13,257 significant SNP-module pairs. The much more signals detected could be due to mappability or data quality.</p>
</div>
<div id="controlling-mappability-or-not-for-muscle" class="section level3">
<h3>3. Controlling mappability or not for muscle</h3>
When not controlling mappability for muscle, I used the normalized TPM from GTEx portal directly to conduct Trans-PCO. To make them comparable, I included the same genes in the analysis. The total number of gene modules is 292 (compared to 301 when controlling mappability).
<div class="figure" style="text-align: center">
<img src="assets/mappability_muscle.png" alt="Control mappability or not for muscle (16_all_vs_eur_ind.R)"  />
<p class="caption">
Control mappability or not for muscle (16_all_vs_eur_ind.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-mappability_muscle">
Past versions of mappability_muscle.png
</button>
</p>
<div id="fig-mappability_muscle" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/e77de6d1b68afef38b7ee1bad3c36419a713bf4d/docs/assets/mappability_muscle.png" target="_blank">e77de6d</a>
</td>
<td>
liangend
</td>
<td>
2023-06-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="other-comparison-for-dgn" class="section level3">
<h3>4. Other comparison for DGN</h3>
<div class="figure" style="text-align: center">
<img src="assets/dgn_comp.png" alt="Control mappability or not for muscle (16_all_vs_eur_ind.R)"  />
<p class="caption">
Control mappability or not for muscle (16_all_vs_eur_ind.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dgn_comp">
Past versions of dgn_comp.png
</button>
</p>
<div id="fig-dgn_comp" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/e77de6d1b68afef38b7ee1bad3c36419a713bf4d/docs/assets/dgn_comp.png" target="_blank">e77de6d</a>
</td>
<td>
liangend
</td>
<td>
2023-06-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="functional-enrichment" class="section level3">
<h3>5. Functional enrichment</h3>
<div class="figure" style="text-align: center">
<img src="assets/chrom_enrich_blood.png" alt="Control mappability or not for muscle (06_trans_enrich.R)"  />
<p class="caption">
Control mappability or not for muscle (06_trans_enrich.R)
</p>
</div>
<div class="figure" style="text-align: center">
<img src="assets/chrom_enrich_muscle.png" alt="Control mappability or not for muscle (06_trans_enrich.R)"  />
<p class="caption">
Control mappability or not for muscle (06_trans_enrich.R)
</p>
</div>
<div class="figure" style="text-align: center">
<img src="assets/chrom_enrich_thyroid.png" alt="Control mappability or not for muscle (06_trans_enrich.R)"  />
<p class="caption">
Control mappability or not for muscle (06_trans_enrich.R)
</p>
</div>
</div>
</div>
<div id="section-1" class="section level2">
<h2>5/1/2023</h2>
<div id="distribution-of-trans-signals" class="section level3">
<h3>1. Distribution of trans signals</h3>
32/199 modules were found to have at least one significant trans signal in GTEx blood. In contrast, 102/166 modules were found to have trans-eQTLs in DGN. Most signals were observed in module 135 with 105 SNPs in GTEx (module 51 with 695 SNPs in DGN).
<div class="figure" style="text-align: center">
<img src="assets/mod_dist.png" alt="Number of significant SNPs across modules (08_mod_dist.R)"  />
<p class="caption">
Number of significant SNPs across modules (08_mod_dist.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-mod_dist">
Past versions of mod_dist.png
</button>
</p>
<div id="fig-mod_dist" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/8b19366b4dd7fdaa63ad2cc6504cc145d633ada4/docs/assets/mod_dist.png" target="_blank">8b19366</a>
</td>
<td>
liangend
</td>
<td>
2023-05-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/f22c4a7610f94d3fa895bf2c643f5896dfe3f2e6/docs/assets/mod_dist.png" target="_blank">f22c4a7</a>
</td>
<td>
liangend
</td>
<td>
2023-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
Most signals were found on chromosome 3. In GTEx blood, there are significant 108 SNPs from chr3 (1237 SNPs from chr3 in DGN). Out of these 108 SNPs, 105 SNPs are relative to module 135 (652/1237 SNPs from chr3 are relative to module 51 in DGN).
<div class="figure" style="text-align: center">
<img src="assets/chr_dist.png" alt="Number of significant SNPs across chromosomes (08_mod_dist.R)"  />
<p class="caption">
Number of significant SNPs across chromosomes (08_mod_dist.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-chr_dist">
Past versions of chr_dist.png
</button>
</p>
<div id="fig-chr_dist" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/8b19366b4dd7fdaa63ad2cc6504cc145d633ada4/docs/assets/chr_dist.png" target="_blank">8b19366</a>
</td>
<td>
liangend
</td>
<td>
2023-05-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/f22c4a7610f94d3fa895bf2c643f5896dfe3f2e6/docs/assets/chr_dist.png" target="_blank">f22c4a7</a>
</td>
<td>
liangend
</td>
<td>
2023-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Genes in module 135 of GTEx: AURKC, HKR1, ZNF134, ZNF135, ZNF211, ZNF256, ZNF304, ZNF329, ZNF347, ZNF418, ZNF543, ZNF544, ZNF548, ZNF549, ZNF550, ZNF551, ZNF585B, ZNF606, ZNF671, ZNF772, ZNF776, ZSCAN18</p>
<p>Genes in module 51 of DGN (61 genes): AURKC, HKR1, LINC00665, PHKB, TRAPPC2P1, ZFP28, ZFP82, ZIK1, ZNF107, ZNF134, ZNF135, ZNF154, ZNF160, ZNF211, ZNF256, ZNF274, ZNF285, ZNF304, ZNF320, ZNF329, ZSCAN18, etc.</p>
<p>20/22 genes in module 135 of GTEx overlaps with module 51 of DGN. Functions of module 135 involves RNA polymerase II transcription regulation and binding, DNA-binding transcription factor activity.</p>
</div>
<div id="replication-of-gtex-trans-signals-based-on-dgn-module" class="section level3">
<h3>2. Replication of GTEx trans signals based on DGN module</h3>
There are 256 significant trans SNP-module pairs (p &lt; 5e-8) found in GTEx blood based on DGN modules. 205 of them were also included in the analysis of DGN, and 150 SNP-module pairs were also significant (p &lt; 5e-8)
<div class="figure" style="text-align: center">
<img src="assets/gtex_dgn_rep.png" alt="Trans signal replication between GTEx and DGN"  />
<p class="caption">
Trans signal replication between GTEx and DGN
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-gtex_dgn_rep">
Past versions of gtex_dgn_rep.png
</button>
</p>
<div id="fig-gtex_dgn_rep" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/72e36c1dbcd8173a6ebf16605e01d8cef4d9b085/docs/assets/gtex_dgn_rep.png" target="_blank">72e36c1</a>
</td>
<td>
liangend
</td>
<td>
2023-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="simulation-of-the-effect-of-genes-and-samples-on-the-detection-of-significant-modules" class="section level3">
<h3>3. Simulation of the effect of # genes and samples on the detection of significant modules</h3>
<p>I simulated a module with 100/200 genes, with one gene causally regulated by a SNP. Heritability of the gene was set to be 0.15. The genotype of the SNP was randomly generated for 10/100/300/500/700/1000/2000 individuals. Expression of other genes were generated from N(0, 1). In total 1000 simulations were conducted for each scenario.</p>
<div class="figure" style="text-align: center">
<img src="assets/null_p_gene100.png" alt="Null p values for gene module with 100 genes (no causal genes were simulated; 07_sim_ana.R)"  />
<p class="caption">
Null p values for gene module with 100 genes (no causal genes were simulated; 07_sim_ana.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-null_p_gene100">
Past versions of null_p_gene100.png
</button>
</p>
<div id="fig-null_p_gene100" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/72e36c1dbcd8173a6ebf16605e01d8cef4d9b085/docs/assets/null_p_gene100.png" target="_blank">72e36c1</a>
</td>
<td>
liangend
</td>
<td>
2023-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/null_p_hist_gene100.png" alt="Null p values for gene module with 100 genes (no causal genes were simulated)"  />
<p class="caption">
Null p values for gene module with 100 genes (no causal genes were simulated)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-null_p_hist_gene100">
Past versions of null_p_hist_gene100.png
</button>
</p>
<div id="fig-null_p_hist_gene100" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/8b19366b4dd7fdaa63ad2cc6504cc145d633ada4/docs/assets/null_p_hist_gene100.png" target="_blank">8b19366</a>
</td>
<td>
liangend
</td>
<td>
2023-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/null_p_gene200.png" alt="Null p values for gene module with 200 genes (no causal genes were simulated)"  />
<p class="caption">
Null p values for gene module with 200 genes (no causal genes were simulated)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-null_p_gene200">
Past versions of null_p_gene200.png
</button>
</p>
<div id="fig-null_p_gene200" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/72e36c1dbcd8173a6ebf16605e01d8cef4d9b085/docs/assets/null_p_gene200.png" target="_blank">72e36c1</a>
</td>
<td>
liangend
</td>
<td>
2023-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/null_p_hist_gene200.png" alt="Null p values for gene module with 200 genes (no causal genes were simulated)"  />
<p class="caption">
Null p values for gene module with 200 genes (no causal genes were simulated)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-null_p_hist_gene200">
Past versions of null_p_hist_gene200.png
</button>
</p>
<div id="fig-null_p_hist_gene200" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/8b19366b4dd7fdaa63ad2cc6504cc145d633ada4/docs/assets/null_p_hist_gene200.png" target="_blank">8b19366</a>
</td>
<td>
liangend
</td>
<td>
2023-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
Type I error rate of PCO ACAT under different scenarios
<div class="figure" style="text-align: center">
<img src="assets/type_I_error_sim.png" alt="Type I error rate"  />
<p class="caption">
Type I error rate
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-type_I_error_sim">
Past versions of type_I_error_sim.png
</button>
</p>
<div id="fig-type_I_error_sim" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/72e36c1dbcd8173a6ebf16605e01d8cef4d9b085/docs/assets/type_I_error_sim.png" target="_blank">72e36c1</a>
</td>
<td>
liangend
</td>
<td>
2023-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
Power of PCO ACAT under different scenarios
<div class="figure" style="text-align: center">
<img src="assets/power_sim.png" alt="Power analysis"  />
<p class="caption">
Power analysis
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-power_sim">
Past versions of power_sim.png
</button>
</p>
<div id="fig-power_sim" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/72e36c1dbcd8173a6ebf16605e01d8cef4d9b085/docs/assets/power_sim.png" target="_blank">72e36c1</a>
</td>
<td>
liangend
</td>
<td>
2023-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The more genes in one module, the more samples required to detect the signal.</p>
</div>
<div id="multiple-testing" class="section level3">
<h3>4. Multiple testing</h3>
<p>Use the method in GTEx to find “eModule”:</p>
<ol style="list-style-type: decimal">
<li><p>Take the most extreme P-value per gene module, and multiply that P-value by 1e6;</p></li>
<li><p>Then apply Benjamini-Hochberg on the adjusted extreme P-values across gene modules to control FDR.</p>
<div class="figure" style="text-align: center">
<img src="assets/n_sig_mod.png" alt="Number of significant modules detected"  />
<p class="caption">
Number of significant modules detected
</p>
</div></li>
</ol>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-n_sig_mod">
Past versions of n_sig_mod.png
</button>
</p>
<div id="fig-n_sig_mod" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/8b19366b4dd7fdaa63ad2cc6504cc145d633ada4/docs/assets/n_sig_mod.png" target="_blank">8b19366</a>
</td>
<td>
liangend
</td>
<td>
2023-05-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/72e36c1dbcd8173a6ebf16605e01d8cef4d9b085/docs/assets/n_sig_mod.png" target="_blank">72e36c1</a>
</td>
<td>
liangend
</td>
<td>
2023-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/n_sig_comp.png" alt="Number of significant SNP-module pairs detected (99_scripts/08_all_sig.R)"  />
<p class="caption">
Number of significant SNP-module pairs detected (99_scripts/08_all_sig.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-n_sig_comp">
Past versions of n_sig_comp.png
</button>
</p>
<div id="fig-n_sig_comp" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/662970654571583f081eedbf17d505ee90d43022/docs/assets/n_sig_comp.png" target="_blank">6629706</a>
</td>
<td>
liangend
</td>
<td>
2023-05-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Based on permutation FDR correction, there are 108 significant SNP-module pairs from 4 chromosomes and 6 modules, 100 of which were between chr3 and mod135 (nominal p-value threshold = 4.6e-9).</p>
</div>
<div id="divide-large-modules-into-smaller-ones" class="section level3">
<h3>5. Divide large modules into smaller ones</h3>
I divided large modules (module size &gt; half of the sample size) into smaller modules (sub module size &lt;= half of the sample size).
<div class="figure" style="text-align: center">
<img src="assets/mod_div_comp.png" alt="Module division comparison (10_mod_div_comp.R)"  />
<p class="caption">
Module division comparison (10_mod_div_comp.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-mod_div_comp">
Past versions of mod_div_comp.png
</button>
</p>
<div id="fig-mod_div_comp" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/ea603198bb309a1cb377b46132e8dd18e8d7971e/docs/assets/mod_div_comp.png" target="_blank">ea60319</a>
</td>
<td>
liangend
</td>
<td>
2023-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="summary-of-11-tissues" class="section level3">
<h3>6. Summary of 11 tissues</h3>
<div class="figure" style="text-align: center">
<img src="assets/basic_stats.png" alt="Summary of each tissue (10_sum_11tissues.R)"  />
<p class="caption">
Summary of each tissue (10_sum_11tissues.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-basic_stats">
Past versions of basic_stats.png
</button>
</p>
<div id="fig-basic_stats" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/8b19366b4dd7fdaa63ad2cc6504cc145d633ada4/docs/assets/basic_stats.png" target="_blank">8b19366</a>
</td>
<td>
liangend
</td>
<td>
2023-05-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/bf2c1ff2465f6c5b94f4e8f8173be70cb0690b17/docs/assets/basic_stats.png" target="_blank">bf2c1ff</a>
</td>
<td>
liangend
</td>
<td>
2023-05-04
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/459110b821e87af81ab3691591e88281e1d6bb74/docs/assets/basic_stats.png" target="_blank">459110b</a>
</td>
<td>
liangend
</td>
<td>
2023-05-04
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/7e91da3768b3c621f7063ec83ea1c96e6b052625/docs/assets/basic_stats.png" target="_blank">7e91da3</a>
</td>
<td>
liangend
</td>
<td>
2023-05-04
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/ea603198bb309a1cb377b46132e8dd18e8d7971e/docs/assets/basic_stats.png" target="_blank">ea60319</a>
</td>
<td>
liangend
</td>
<td>
2023-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/sig_summary.png" alt="Significant modules and SNPs (10_sum_11tissues.R)"  />
<p class="caption">
Significant modules and SNPs (10_sum_11tissues.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-sig_summary">
Past versions of sig_summary.png
</button>
</p>
<div id="fig-sig_summary" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/8b19366b4dd7fdaa63ad2cc6504cc145d633ada4/docs/assets/sig_summary.png" target="_blank">8b19366</a>
</td>
<td>
liangend
</td>
<td>
2023-05-09
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/ea603198bb309a1cb377b46132e8dd18e8d7971e/docs/assets/sig_summary.png" target="_blank">ea60319</a>
</td>
<td>
liangend
</td>
<td>
2023-05-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
Most significant pairs were observed from testis, similar as GTEx finding the most trans signals from testis. In total 3361 significant SNP-module pairs found from 28 modules. However, most signals were found between module 168 and chr17 (2855 pairs). Comparing to trans-eGenes found in GTEx (49 genes), 28 modules contains 959 genes, 19 of which overlapped with GTEx ones.
<div class="figure" style="text-align: center">
<img src="assets/snp_dist_testis.png" alt="Number of significant SNPs across modules (08_mod_dist.R)"  />
<p class="caption">
Number of significant SNPs across modules (08_mod_dist.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-snp_dist_testis">
Past versions of snp_dist_testis.png
</button>
</p>
<div id="fig-snp_dist_testis" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/f88dbdcea6cfbe56f22af46cb7e95945c72d170b/docs/assets/snp_dist_testis.png" target="_blank">f88dbdc</a>
</td>
<td>
liangend
</td>
<td>
2023-05-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="using-snps-with-maf-0.01-for-muscle" class="section level3">
<h3>7. Using SNPs with MAF &gt; 0.01 for muscle</h3>
6,380,580 SNPs with MAF &gt; 0.05; 10,403,248 SNPs with MAF &gt; 0.01.
<div class="figure" style="text-align: center">
<img src="assets/maf_comp.png" alt="Module FDR and significant SNP-module pairs (11_maf_comp.R)"  />
<p class="caption">
Module FDR and significant SNP-module pairs (11_maf_comp.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-maf_comp">
Past versions of maf_comp.png
</button>
</p>
<div id="fig-maf_comp" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/8b19366b4dd7fdaa63ad2cc6504cc145d633ada4/docs/assets/maf_comp.png" target="_blank">8b19366</a>
</td>
<td>
liangend
</td>
<td>
2023-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="trans-signals-overlapping-with-cis-signals" class="section level3">
<h3>8. Trans signals overlapping with cis signals</h3>
Across all 11 tissues, on average 63.3% of trans signal overlap with cis-eQTLs; 25.2% overlap with cis-sQTLs; 64.9% overlap with either cis-eQTLs or cis-sQTLs.
<div class="figure" style="text-align: center">
<img src="assets/overlap_cis.png" alt="Overlapping with cis signals (12_overlap_cis.R)"  />
<p class="caption">
Overlapping with cis signals (12_overlap_cis.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-overlap_cis">
Past versions of overlap_cis.png
</button>
</p>
<div id="fig-overlap_cis" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/2af74acfe09c5c7335e27b0d4b9e6f9fd74dc7e8/docs/assets/overlap_cis.png" target="_blank">2af74ac</a>
</td>
<td>
liangend
</td>
<td>
2023-05-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/f88dbdcea6cfbe56f22af46cb7e95945c72d170b/docs/assets/overlap_cis.png" target="_blank">f88dbdc</a>
</td>
<td>
liangend
</td>
<td>
2023-05-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
Across 6 tissues with data of eQTL, sQTL, mQTL and RNA editing, mQTL did not add too much extra information.
<div class="figure" style="text-align: center">
<img src="assets/overlap_cis2.png" alt="Overlapping with cis signals (12_overlap_cis.R)"  />
<p class="caption">
Overlapping with cis signals (12_overlap_cis.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-overlap_cis2">
Past versions of overlap_cis2.png
</button>
</p>
<div id="fig-overlap_cis2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/cef86d52aad8e5bd99699f963f164a39cd1f59f7/docs/assets/overlap_cis2.png" target="_blank">cef86d5</a>
</td>
<td>
liangend
</td>
<td>
2023-06-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="colocalization-with-gwas-signals" class="section level3">
<h3>9. Colocalization with GWAS signals</h3>
Trans-eQTL loci are defined as 100kb flanking regions around leading SNPs.
<div class="figure" style="text-align: center">
<img src="assets/gwas_coloc.png" alt="Colocalization with 4 GWAS traits (13_gwas_coloc.R)"  />
<p class="caption">
Colocalization with 4 GWAS traits (13_gwas_coloc.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-gwas_coloc">
Past versions of gwas_coloc.png
</button>
</p>
<div id="fig-gwas_coloc" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/2af74acfe09c5c7335e27b0d4b9e6f9fd74dc7e8/docs/assets/gwas_coloc.png" target="_blank">2af74ac</a>
</td>
<td>
liangend
</td>
<td>
2023-05-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/f88dbdcea6cfbe56f22af46cb7e95945c72d170b/docs/assets/gwas_coloc.png" target="_blank">f88dbdc</a>
</td>
<td>
liangend
</td>
<td>
2023-05-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
Colocalization with 29 blood relevant traits (red blood cell count, platelet count, neutrophil count, etc.) and 3 autoimmune diseases (ulcerative colitis, Crohn’s disease and inflammatory bowel disease). Testis contained the most colocalization (27 traits with at least one colocalization), followed by skin (18), blood (15), nerve (11) and thyroid (10). But blood had the highest colocalization proportion (# cololization / # of trans-eQTL loci). “Average” is the average proportion of of coloc across all traits, while “Unique” is the total number of unique loci colocalized with at least one GWAS signal across all traits.
<div class="figure" style="text-align: center">
<img src="assets/blood_coloc_prop.png" alt="Colocalization with blood and autoimmune disease traits (13_gwas_coloc.R)"  />
<p class="caption">
Colocalization with blood and autoimmune disease traits (13_gwas_coloc.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-blood_coloc_prop">
Past versions of blood_coloc_prop.png
</button>
</p>
<div id="fig-blood_coloc_prop" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/662970654571583f081eedbf17d505ee90d43022/docs/assets/blood_coloc_prop.png" target="_blank">6629706</a>
</td>
<td>
liangend
</td>
<td>
2023-05-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/c0c502cf9f3d94484d881ee5c4587b74ed8b0613/docs/assets/blood_coloc_prop.png" target="_blank">c0c502c</a>
</td>
<td>
liangend
</td>
<td>
2023-05-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
I did the heritability enrichment analysis for one colocalized signal of trans-eQTL (on chr3 associated with module 7) in blood. Among 32 traits, h2 enrichment of 22 are shown below (p &lt; 0.05). 6 of them reached significance after Bonferroni correction, and 8 of them have a corresponding colocalization signal (PP4 &gt; 0.5).
<div class="figure" style="text-align: center">
<img src="assets/ldsc_blood_coloc.png" alt="Heritability enrichment of trans signal on module 7 (14_h2_enrich.R)"  />
<p class="caption">
Heritability enrichment of trans signal on module 7 (14_h2_enrich.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ldsc_blood_coloc">
Past versions of ldsc_blood_coloc.png
</button>
</p>
<div id="fig-ldsc_blood_coloc" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/797c642011b324959a4ece7394cc29005e8d0a12/docs/assets/ldsc_blood_coloc.png" target="_blank">797c642</a>
</td>
<td>
liangend
</td>
<td>
2023-05-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
For example, I identified an interesting colocalization between MPV (mean platelet volume) associated locus near ARHGEF3 and a trans-eQTL of module 7, which associated with platelet activation and hemostasis (PP4 = 0.68).
<div class="figure" style="text-align: center">
<img src="assets/coloc_MPV_blood.png" alt="Colocalization between MPV GWAS and trans signal on module 7 (14_h2_enrich.R)"  />
<p class="caption">
Colocalization between MPV GWAS and trans signal on module 7 (14_h2_enrich.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-coloc_MPV_blood">
Past versions of coloc_MPV_blood.png
</button>
</p>
<div id="fig-coloc_MPV_blood" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/797c642011b324959a4ece7394cc29005e8d0a12/docs/assets/coloc_MPV_blood.png" target="_blank">797c642</a>
</td>
<td>
liangend
</td>
<td>
2023-05-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="model-calibration" class="section level3">
<h3>10. Model calibration</h3>
<p>Given the limited number of significant SNPs I observe, I tried to include different number of covariates in the model to see which gives the most significant signals. I tried 4 different scenarios on muscle: 1) 60 expression PCs + 5 genotype PCs + 3 covariates; 2) 10 expression PCs + 5 genotype PCs + 3 covariates; 3) 5 genotype PCs + 3 covariates; 4) No covariates.</p>
<div class="figure" style="text-align: center">
<img src="assets/qq_obs_muscle_chr22.png" alt="QQ plot of p values for chr22 (15_pval_calib.R)"  />
<p class="caption">
QQ plot of p values for chr22 (15_pval_calib.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-qq_obs_muscle_chr22">
Past versions of qq_obs_muscle_chr22.png
</button>
</p>
<div id="fig-qq_obs_muscle_chr22" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/cef86d52aad8e5bd99699f963f164a39cd1f59f7/docs/assets/qq_obs_muscle_chr22.png" target="_blank">cef86d5</a>
</td>
<td>
liangend
</td>
<td>
2023-06-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Number of significant eModules and eSNPs:</p>
<ol style="list-style-type: decimal">
<li><p>68 covaraites: 33 eModules, 417 eSNPs;</p></li>
<li><p>18 covaraites: 10 eModules, 180 eSNPs;</p></li>
<li><p>8 covaraites: 3 eModules, 80 eSNPs;</p></li>
<li><p>No covaraites: 274 eModules, 2,430,382 eSNPs.</p></li>
</ol>
Null p distribution
<div class="figure" style="text-align: center">
<img src="assets/qq_muscle_chr20.png" alt="QQ plot of p values for chr20 (15_pval_calib.R)"  />
<p class="caption">
QQ plot of p values for chr20 (15_pval_calib.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-qq_muscle_chr20">
Past versions of qq_muscle_chr20.png
</button>
</p>
<div id="fig-qq_muscle_chr20" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/cef86d52aad8e5bd99699f963f164a39cd1f59f7/docs/assets/qq_muscle_chr20.png" target="_blank">cef86d5</a>
</td>
<td>
liangend
</td>
<td>
2023-06-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/histP_muscle_chr20.png" alt="Histogram of p values for chr20 (15_pval_calib.R)"  />
<p class="caption">
Histogram of p values for chr20 (15_pval_calib.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-histP_muscle_chr20">
Past versions of histP_muscle_chr20.png
</button>
</p>
<div id="fig-histP_muscle_chr20" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/cef86d52aad8e5bd99699f963f164a39cd1f59f7/docs/assets/histP_muscle_chr20.png" target="_blank">cef86d5</a>
</td>
<td>
liangend
</td>
<td>
2023-06-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/qq_muscle_chr21.png" alt="QQ plot of p values for chr21 (15_pval_calib.R)"  />
<p class="caption">
QQ plot of p values for chr21 (15_pval_calib.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-qq_muscle_chr21">
Past versions of qq_muscle_chr21.png
</button>
</p>
<div id="fig-qq_muscle_chr21" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/cef86d52aad8e5bd99699f963f164a39cd1f59f7/docs/assets/qq_muscle_chr21.png" target="_blank">cef86d5</a>
</td>
<td>
liangend
</td>
<td>
2023-06-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/histP_muscle_chr21.png" alt="Histogram of p values for chr21 (15_pval_calib.R)"  />
<p class="caption">
Histogram of p values for chr21 (15_pval_calib.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-histP_muscle_chr21">
Past versions of histP_muscle_chr21.png
</button>
</p>
<div id="fig-histP_muscle_chr21" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/cef86d52aad8e5bd99699f963f164a39cd1f59f7/docs/assets/histP_muscle_chr21.png" target="_blank">cef86d5</a>
</td>
<td>
liangend
</td>
<td>
2023-06-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/qq_muscle_chr22.png" alt="QQ plot of p values for chr22 (15_pval_calib.R)"  />
<p class="caption">
QQ plot of p values for chr22 (15_pval_calib.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-qq_muscle_chr22">
Past versions of qq_muscle_chr22.png
</button>
</p>
<div id="fig-qq_muscle_chr22" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/cef86d52aad8e5bd99699f963f164a39cd1f59f7/docs/assets/qq_muscle_chr22.png" target="_blank">cef86d5</a>
</td>
<td>
liangend
</td>
<td>
2023-06-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/histP_muscle_chr22.png" alt="Histogram of p values for chr22 (15_pval_calib.R)"  />
<p class="caption">
Histogram of p values for chr22 (15_pval_calib.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-histP_muscle_chr22">
Past versions of histP_muscle_chr22.png
</button>
</p>
<div id="fig-histP_muscle_chr22" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/cef86d52aad8e5bd99699f963f164a39cd1f59f7/docs/assets/histP_muscle_chr22.png" target="_blank">cef86d5</a>
</td>
<td>
liangend
</td>
<td>
2023-06-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="section-2" class="section level2">
<h2>4/12/2023</h2>
<div id="new-read-count-results-based-on-adjusted-gencode-gtf-for-gtex-v8" class="section level3">
<h3>1. New read count results based on adjusted GENCODE gtf (for GTEx V8)</h3>
<p>After QC and removing genes on chrX, chrM and chrY, 12929 genes for 670 individuals were kept. Among these genes, 12039 are protein coding genes and lincRNA.</p>
<div id="compare-my-tpm-with-gtex-tpm-including-genes-on-chrx-chrm-and-chry" class="section level4">
<h4>Compare my TPM with GTEx TPM (including genes on chrX, chrM and chrY)</h4>
14047 genes with cor &gt; 0.85; 32 genes with cor &lt;= 0.85
<div class="figure" style="text-align: center">
<img src="assets/tpm_cor.png" alt="Cor between GTEx TPM and my TPM"  />
<p class="caption">
Cor between GTEx TPM and my TPM
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-tpm_cor">
Past versions of tpm_cor.png
</button>
</p>
<div id="fig-tpm_cor" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/69afe07f20d02c7615f56385d75f2ef177640448/docs/assets/tpm_cor.png" target="_blank">69afe07</a>
</td>
<td>
liangend
</td>
<td>
2023-03-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/gene_tpm.png" alt="TPM of selected genes"  />
<p class="caption">
TPM of selected genes
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-gene_tpm">
Past versions of gene_tpm.png
</button>
</p>
<div id="fig-gene_tpm" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/69afe07f20d02c7615f56385d75f2ef177640448/docs/assets/gene_tpm.png" target="_blank">69afe07</a>
</td>
<td>
liangend
</td>
<td>
2023-03-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
13196 genes with mean TPM &lt; 100 for both groups.
<div class="figure" style="text-align: center">
<img src="assets/mean_tpm.png" alt="Mean TPM across 670 individuals"  />
<p class="caption">
Mean TPM across 670 individuals
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-mean_tpm">
Past versions of mean_tpm.png
</button>
</p>
<div id="fig-mean_tpm" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/69afe07f20d02c7615f56385d75f2ef177640448/docs/assets/mean_tpm.png" target="_blank">69afe07</a>
</td>
<td>
liangend
</td>
<td>
2023-03-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="compare-pc-with-peer" class="section level3">
<h3>2. Compare PC with PEER</h3>
For the normalized TPM, the first 60 PCs and PEER factors are highly correlated.
<div class="figure" style="text-align: center">
<img src="assets/pc_peer_cor.png" alt="Correlation between PC and PEER factor"  />
<p class="caption">
Correlation between PC and PEER factor
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-pc_peer_cor">
Past versions of pc_peer_cor.png
</button>
</p>
<div id="fig-pc_peer_cor" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/6514442cb5408edf879fa24e1ca3e5f689621b58/docs/assets/pc_peer_cor.png" target="_blank">6514442</a>
</td>
<td>
liangend
</td>
<td>
2023-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Zhou et al. (2022) suggested that PCA performs close to or better than PEER factors for QTL mapping under different scenarios, but with much shorter time and better interpretability.</p>
</div>
<div id="coexpression-analysis" class="section level3">
<h3>3. Coexpression analysis</h3>
<p>I used 5 genotype PCs, 3 factors (sex, pcr, platform), and different numbers (10 and 60) of expression PCs or PEER factors as the covariates to obtain the residual expression, which is further used as the input of WGCNA.</p>
I used the elbow method (Zhou et al., 2022) the optimized the number of PCs to be included.
<div class="figure" style="text-align: center">
<img src="assets/pc_elbow.png" alt="Optimize the number of PCs using the elbow method"  />
<p class="caption">
Optimize the number of PCs using the elbow method
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-pc_elbow">
Past versions of pc_elbow.png
</button>
</p>
<div id="fig-pc_elbow" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/6514442cb5408edf879fa24e1ca3e5f689621b58/docs/assets/pc_elbow.png" target="_blank">6514442</a>
</td>
<td>
liangend
</td>
<td>
2023-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Another optimization method, BE algorithm, tends to include more PCs than the elbow method, but both are fewer the number of PEER factors included in the GTEx paper.</p>
<p>Number of modules, unassigned genes, and genes in top 3 modules when including different number of PCs or PEER factors. <img src="assets/coexp_comp.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-coexp_comp">
Past versions of coexp_comp.png
</button>
</p>
<div id="fig-coexp_comp" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/6514442cb5408edf879fa24e1ca3e5f689621b58/docs/assets/coexp_comp.png" target="_blank">6514442</a>
</td>
<td>
liangend
</td>
<td>
2023-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/wgcna_dendro_19pc.png" alt="Cluster dendrogram of genes when including 19 PCs"  />
<p class="caption">
Cluster dendrogram of genes when including 19 PCs
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-wgcna_dendro_19pc">
Past versions of wgcna_dendro_19pc.png
</button>
</p>
<div id="fig-wgcna_dendro_19pc" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/6514442cb5408edf879fa24e1ca3e5f689621b58/docs/assets/wgcna_dendro_19pc.png" target="_blank">6514442</a>
</td>
<td>
liangend
</td>
<td>
2023-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>For trans-PCO analysis, I conducted co-expression analysis based on 5 genotype PCs, 3 factors (sex, pcr, platform), and 10 expression PCs, which resulted in 199 modules, 235 unassigned genes, and top 3 modules containing 858, 481, and 475 genes.</p>
<div id="functional-profiling-of-gene-modules" class="section level4">
<h4>Functional profiling of gene modules</h4>
To check the sanity of gene module classification, I investigated the enrichment of GO terms (FDR &lt; 0.05) for several gene modules using g:Profiler.
<div class="figure" style="text-align: center">
<img src="assets/mod_go.png" alt="Shared and unique GO terms for module 5, 6 and 7"  />
<p class="caption">
Shared and unique GO terms for module 5, 6 and 7
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-mod_go">
Past versions of mod_go.png
</button>
</p>
<div id="fig-mod_go" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/ac9a8f38e42aa6677cab1a8a396840b833e26ff8/docs/assets/mod_go.png" target="_blank">ac9a8f3</a>
</td>
<td>
liangend
</td>
<td>
2023-04-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Top 3 significant BP terms:</p>
<p>module 5 (313 genes): leukocyte activation, cell activation, lymphocyte activation;</p>
<p>module 6 (312 genes): defense response to symbiont, defense response to virus, response to virus;</p>
<p>module 7 (307 genes): wound healing, platelet activation, blood coagulation.</p>
Another example:
<div class="figure" style="text-align: center">
<img src="assets/mod_go2.png" alt="Shared and unique GO terms for module 106, 107 and 108"  />
<p class="caption">
Shared and unique GO terms for module 106, 107 and 108
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-mod_go2">
Past versions of mod_go2.png
</button>
</p>
<div id="fig-mod_go2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/ac9a8f38e42aa6677cab1a8a396840b833e26ff8/docs/assets/mod_go2.png" target="_blank">ac9a8f3</a>
</td>
<td>
liangend
</td>
<td>
2023-04-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Top 3 significant BP terms:</p>
<p>module 106 (31 genes): extracellular structure organization, extracellular matrix organization, external encapsulating structure organization;</p>
<p>module 107 (31 genes): monosaccharide metabolic process, hexose metabolic process, glucose metabolic process;</p>
<p>module 108 (31 genes): None</p>
</div>
</div>
<div id="cis-eqtl-analysis" class="section level3">
<h3>4. cis-eQTL analysis</h3>
<p>Included 5 genotype PCs, 3 factors (sex, pcr, platform), 10 expression PCs as covariates.</p>
<p>Genotype data: 46,569,704 variants after QC. 10,726,114 SNPs (MAF &gt;= 0.01) were included in the cis-eQTL analysis.</p>
12360 eGenes (qval &lt; 0.05) in GTEx data; 6067 eGenes in my analysis. Among 12360 eGenes reported by GTEx, there are 3858 genes not included in our analysis.
<div class="figure" style="text-align: center">
<img src="assets/egene_venn.png" alt="Comparison of number of eGenes"  />
<p class="caption">
Comparison of number of eGenes
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-egene_venn">
Past versions of egene_venn.png
</button>
</p>
<div id="fig-egene_venn" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/100cd7d6e431fcb38f83c29ebb1a02495853d3eb/docs/assets/egene_venn.png" target="_blank">100cd7d</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/20953797f627a906fe38e3def800262bdb5ee493/docs/assets/egene_venn.png" target="_blank">2095379</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
1906 same top variants associated with the same eGene. Slope cor = 0.99
<div class="figure" style="text-align: center">
<img src="assets/cis_eqtl_slope_comp.png" alt="Comparison of the slopes of shared variants"  />
<p class="caption">
Comparison of the slopes of shared variants
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-cis_eqtl_slope_comp">
Past versions of cis_eqtl_slope_comp.png
</button>
</p>
<div id="fig-cis_eqtl_slope_comp" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/a4d7311dbaaccf37ffa802197715bc03dcf5cf87/docs/assets/cis_eqtl_slope_comp.png" target="_blank">a4d7311</a>
</td>
<td>
liangend
</td>
<td>
2023-03-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>When using the same covarites (60 PEER factors) as GTEx, the number of eGenes increases to 8320.</p>
</div>
<div id="trans-eqtl-analysis" class="section level3">
<h3>5. trans-eQTL analysis</h3>
<p>Included 5 genotype PCs, 3 factors (sex, pcr, platform), 60 expression PCs as covariates.</p>
<p>Genotype data: 6,380,580 SNPs (MAF &gt;= 0.05; 6.2 million SNPs in DGN) were included in the trans-eQTL analysis.</p>
<div id="trans-eqtl-mapping-based-on-166-gene-modules-from-wang-et-al.-2022" class="section level4">
<h4>trans-eQTL mapping based on 166 gene modules from Wang et al. (2022)</h4>
In trans-eQTL mapping for DGN, there are 3899 significant trans-eQTL SNP–module pairs (Wang et al., 2022). For the 12132 genes (166 gene modules) included in Wang et al. (2022), there are 10547 genes included in my analysis. Among 3330 shared SNP-gene module pairs, 470 (14.0%) were replicated in my analysis (p &lt; 0.05/3330).
<div class="figure" style="text-align: center">
<img src="assets/trans_sig_rep.png" alt="Trans-eQTL replication for DGN"  />
<p class="caption">
Trans-eQTL replication for DGN
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-trans_sig_rep">
Past versions of trans_sig_rep.png
</button>
</p>
<div id="fig-trans_sig_rep" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/c94e9f8620d467ff350d330beb962b067083cc72/docs/assets/trans_sig_rep.png" target="_blank">c94e9f8</a>
</td>
<td>
liangend
</td>
<td>
2023-04-10
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/ac9a8f38e42aa6677cab1a8a396840b833e26ff8/docs/assets/trans_sig_rep.png" target="_blank">ac9a8f3</a>
</td>
<td>
liangend
</td>
<td>
2023-04-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="trans-eqtl-mapping-based-on-199-gene-modules" class="section level4">
<h4>trans-eQTL mapping based on 199 gene modules</h4>
In total 159 significant signals were found in GTEx data (p &lt; 5e-8). I then investigated the regulatory activity of these signals and those from Wang et al. (2022) based on chromatin states of neutrophils from peripheral blood (Roadmap project).
<div class="figure" style="text-align: center">
<img src="assets/trans_func_enrich.png" alt="Functional enrichment of significant trans signals"  />
<p class="caption">
Functional enrichment of significant trans signals
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-trans_func_enrich">
Past versions of trans_func_enrich.png
</button>
</p>
<div id="fig-trans_func_enrich" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/c94e9f8620d467ff350d330beb962b067083cc72/docs/assets/trans_func_enrich.png" target="_blank">c94e9f8</a>
</td>
<td>
liangend
</td>
<td>
2023-04-10
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/ac9a8f38e42aa6677cab1a8a396840b833e26ff8/docs/assets/trans_func_enrich.png" target="_blank">ac9a8f3</a>
</td>
<td>
liangend
</td>
<td>
2023-04-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
GTEx V8 reported 13 trans-eQTLs in whole blood (Wang et al., 2022), and 4 (30.8%) were replicated in my analysis (p &lt; 0.05/13).
<div class="figure" style="text-align: center">
<img src="assets/trans_gtex_rep.png" alt="Trans-eQTL replication for GTEx"  />
<p class="caption">
Trans-eQTL replication for GTEx
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-trans_gtex_rep">
Past versions of trans_gtex_rep.png
</button>
</p>
<div id="fig-trans_gtex_rep" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/9856c4b6cdc2be402efd16ea7130265ad5e8ca43/docs/assets/trans_gtex_rep.png" target="_blank">9856c4b</a>
</td>
<td>
liangend
</td>
<td>
2023-04-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="section-3" class="section level2">
<h2>3/8/2023</h2>
<div id="comparing-the-p-values-of-pco-and-acat-on-lilis-demo" class="section level3">
<h3>1. Comparing the p values of PCO and ACAT on Lili’s demo</h3>
<p>Number of variant-module associations = 136,904. Correlation of two p values = 0.77</p>
Correlation of two p values (both &lt; 0.05) = 0.73
<div class="figure" style="text-align: center">
<img src="assets/pco_acat_comp.png" alt="Comparing the p values of PCO and ACAT"  />
<p class="caption">
Comparing the p values of PCO and ACAT
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-pco_acat_comp">
Past versions of pco_acat_comp.png
</button>
</p>
<div id="fig-pco_acat_comp" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/da48d1deacc7276d19f042d760b2c0c8afce47ce/docs/assets/pco_acat_comp.png" target="_blank">da48d1d</a>
</td>
<td>
liangend
</td>
<td>
2023-03-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="section-4" class="section level2">
<h2>2/21/2023</h2>
<div id="summary" class="section level3">
<h3>1. Summary</h3>
<p>Use GTEx data to generate a whole-genome Trans-eQTL map based on TransPCO.</p>
</div>
<div id="use-whole-blood-data-as-the-initial-test" class="section level3">
<h3>2. Use whole blood data as the initial test</h3>
<ul>
<li><p>670 samples of RNAseq</p></li>
<li><p>26584 genes</p></li>
<li><p>Filter by mapping quality = 255 (uniquely mapping reads; <a href="https://physiology.med.cornell.edu/faculty/skrabanek/lab/angsd/lecture_notes/STARmanual.pdf">STAR mapping manual</a>)</p></li>
<li><p>More strict way to count read <img src="assets/count_modes.png" style="width:90.0%" /></p></li>
<li><p>Filter by CPM (counts per million)</p></li>
</ul>
For each gene, count the number of samples with CPM &gt; 0.5. 11771 genes kept if filter by number of samples with CPM &gt; 0.5 greater than 10% of total samples.
<div class="figure" style="text-align: center">
<img src="assets/Fig1.png" alt="Figure 1: Number of genes kept with different CPM filtering criteria."  />
<p class="caption">
Figure 1: Number of genes kept with different CPM filtering criteria.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Fig1">
Past versions of Fig1.png
</button>
</p>
<div id="fig-Fig1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/543bdfdc3e222209983620b0418d30a73d9a7995/docs/assets/Fig1.png" target="_blank">543bdfd</a>
</td>
<td>
liangend
</td>
<td>
2023-02-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Normalize RPKM (Reads per kilo base of transcript per million)
<div class="figure" style="text-align: center">
<img src="assets/Fig2.png" alt="Figure 2: Density of normalized RPKM of randomly selected 10 genes." width="80%" />
<p class="caption">
Figure 2: Density of normalized RPKM of randomly selected 10 genes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Fig2">
Past versions of Fig2.png
</button>
</p>
<div id="fig-Fig2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/543bdfdc3e222209983620b0418d30a73d9a7995/docs/assets/Fig2.png" target="_blank">543bdfd</a>
</td>
<td>
liangend
</td>
<td>
2023-02-22
</td>
</tr>
</tbody>
</table>
</div>
</div></li>
</ul>
The problematic left tails are caused by too many ties (0’s) for genes that are only found in few samples.
<div class="figure" style="text-align: center">
<img src="assets/Fig3.png" alt="Figure 3: Density of normalized RPKM and original counts of 3 example genes." width="80%" />
<p class="caption">
Figure 3: Density of normalized RPKM and original counts of 3 example genes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Fig3">
Past versions of Fig3.png
</button>
</p>
<div id="fig-Fig3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/bc8d73316f3ad74152295fcd22e49dc47648185e/docs/assets/Fig3.png" target="_blank">bc8d733</a>
</td>
<td>
liangend
</td>
<td>
2023-02-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
When filtering by number of samples with CPM &gt; 0.5 greater than 20% of total samples (10003 genes kept).
<div class="figure" style="text-align: center">
<img src="assets/Fig4.png" alt="Figure 4: Density of normalized RPKM of randomly selected 10 genes using 20% of samples as cutoff." width="80%" />
<p class="caption">
Figure 4: Density of normalized RPKM of randomly selected 10 genes using 20% of samples as cutoff.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Fig4">
Past versions of Fig4.png
</button>
</p>
<div id="fig-Fig4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/dcbbd1312b749700e9e248ec6f5ab47abac942cb/docs/assets/Fig4.png" target="_blank">dcbbd13</a>
</td>
<td>
liangend
</td>
<td>
2023-02-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: CentOS Linux 8

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el8-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8 LC_NUMERIC=C         LC_TIME=C           
 [4] LC_COLLATE=C         LC_MONETARY=C        LC_MESSAGES=C       
 [7] LC_PAPER=C           LC_NAME=C            LC_ADDRESS=C        
[10] LC_TELEPHONE=C       LC_MEASUREMENT=C     LC_IDENTIFICATION=C 

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.6.2

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.10       rstudioapi_0.13   whisker_0.4       knitr_1.38       
 [5] magrittr_2.0.3    R6_2.5.1          rlang_1.1.1       fansi_1.0.4      
 [9] highr_0.9         stringr_1.5.0     tools_4.1.0       xfun_0.30        
[13] utf8_1.2.3        cli_3.6.1         git2r_0.28.0      jquerylib_0.1.4  
[17] htmltools_0.5.1.1 rprojroot_2.0.2   yaml_2.2.1        digest_0.6.27    
[21] tibble_3.2.1      lifecycle_1.0.3   later_1.2.0       sass_0.4.0       
[25] vctrs_0.6.2       promises_1.2.0.1  fs_1.5.0          glue_1.6.2       
[29] evaluate_0.15     rmarkdown_2.13    stringi_1.7.12    bslib_0.2.5.1    
[33] compiler_4.1.0    pillar_1.9.0      jsonlite_1.7.2    httpuv_1.6.1     
[37] pkgconfig_2.0.3  </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
