<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>pQTL.knit</title>

<script src="site_libs/header-attrs-2.13/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Research Progress</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Progress
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="2022.html">2022</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/wflowr_template">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-09-11
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>wflowr_template/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220610code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20220610)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220610code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220610)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliangendTransWIPtree24062e40d3980e6237f77927d8edca6e41261ddetargetblank24062e4a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/liangend/Trans_WIP/tree/24062e40d3980e6237f77927d8edca6e41261dde" target="_blank">24062e4</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomliangendTransWIPtree24062e40d3980e6237f77927d8edca6e41261ddetargetblank24062e4a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/liangend/Trans_WIP/tree/24062e40d3980e6237f77927d8edca6e41261dde" target="_blank">24062e4</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
working directory clean
</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/pQTL.Rmd</code>) and HTML (<code>docs/pQTL.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/24062e40d3980e6237f77927d8edca6e41261dde/docs/pQTL.html" target="_blank">24062e4</a>
</td>
<td>
liangend
</td>
<td>
2023-09-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/06a43fc5ae5650db4cfd796a53fd64e7c690ceed/docs/pQTL.html" target="_blank">06a43fc</a>
</td>
<td>
liangend
</td>
<td>
2023-09-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/cbcb50dcb976e363e3970ec15ea09e68d4903a17/analysis/pQTL.Rmd" target="_blank">cbcb50d</a>
</td>
<td>
liangend
</td>
<td>
2023-09-11
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/812ad3ab5db228ba28b93bfd5ad650224acd66ed/docs/pQTL.html" target="_blank">812ad3a</a>
</td>
<td>
liangend
</td>
<td>
2023-09-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/87f03f8f7d4e9c76688f3f2990576d50f58042af/analysis/pQTL.Rmd" target="_blank">87f03f8</a>
</td>
<td>
liangend
</td>
<td>
2023-09-06
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/7842a66f13459fe9a8bc88b12b304cdd2ee82701/docs/pQTL.html" target="_blank">7842a66</a>
</td>
<td>
liangend
</td>
<td>
2023-08-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/7078b9a18f87245de0013d3be8e9fe2e85792843/analysis/pQTL.Rmd" target="_blank">7078b9a</a>
</td>
<td>
liangend
</td>
<td>
2023-08-30
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/7a6a69027f15d99dc3c24f98cdd56cf63c1f8fa0/docs/pQTL.html" target="_blank">7a6a690</a>
</td>
<td>
liangend
</td>
<td>
2023-08-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/62099ebc325d8c109a282c50e09f1b83dd7a7069/analysis/pQTL.Rmd" target="_blank">62099eb</a>
</td>
<td>
liangend
</td>
<td>
2023-08-28
</td>
<td>
commit message
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/7e2d44663ca5a1dcc9ec12b8dd6a5d371f656965/docs/pQTL.html" target="_blank">7e2d446</a>
</td>
<td>
liangend
</td>
<td>
2023-08-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/83bcadf0b378f062842da99863cf8473554738b7/analysis/pQTL.Rmd" target="_blank">83bcadf</a>
</td>
<td>
liangend
</td>
<td>
2023-08-09
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/4b9038cb654678fff1f14c13a6735064dc9f9a0f/docs/pQTL.html" target="_blank">4b9038c</a>
</td>
<td>
liangend
</td>
<td>
2023-08-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/b71b01774dfad7898d5f32e04b736b3591b70b15/analysis/pQTL.Rmd" target="_blank">b71b017</a>
</td>
<td>
liangend
</td>
<td>
2023-08-07
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/2c91aa57aecb875f3a4ef08473acebfca11aefe2/docs/pQTL.html" target="_blank">2c91aa5</a>
</td>
<td>
liangend
</td>
<td>
2023-08-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/cf2f5bae84a3b0a809f5fe23b35e82d2c500c30c/analysis/pQTL.Rmd" target="_blank">cf2f5ba</a>
</td>
<td>
liangend
</td>
<td>
2023-08-02
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liangend/Trans_WIP/d2b879ee78ef0224a6306d7e1354427d4e57e971/docs/pQTL.html" target="_blank">d2b879e</a>
</td>
<td>
liangend
</td>
<td>
2023-08-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/9e0f8babdfcdcb0c358174545b6b0d5ef76043ff/analysis/pQTL.Rmd" target="_blank">9e0f8ba</a>
</td>
<td>
liangend
</td>
<td>
2023-08-02
</td>
<td>
update
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
</style>
<div id="section" class="section level2">
<h2>9/13/2023</h2>
<div id="enhancer-length-of-pqtl-genes" class="section level3">
<h3>Enhancer length of pQTL genes</h3>
Genes with cis signals have longer enhancer regions then those without cis signals. Error bars were generated based on bootstrapping for 1000 samples. Gene length is positively correlated with pLI (cor = 0.16), while enhancer length is not correlated with enhancer length or pLI.
<div class="figure" style="text-align: center">
<img src="assets/enh_length.png" alt="Enhancer length for pQTL genes (12_enh_length.R)"  />
<p class="caption">
Enhancer length for pQTL genes (12_enh_length.R)
</p>
</div>
<div class="figure" style="text-align: center">
<img src="assets/pLI_enh_gene_len.png" alt="Enhancer length, gene length and pLI for genes with at least one enhancer annotation (12_enh_length.R)"  />
<p class="caption">
Enhancer length, gene length and pLI for genes with at least one enhancer annotation (12_enh_length.R)
</p>
</div>
</div>
</div>
<div id="section-1" class="section level2">
<h2>9/6/2023</h2>
<div id="enhancer-domain-score-eds-of-pqtl-genes" class="section level3">
<h3>Enhancer domain score (EDS) of pQTL genes</h3>
No EDS differences were found between cis-pQTL and trans-pQTL genes
<div class="figure" style="text-align: center">
<img src="assets/pqtl_eds.png" alt="EDS of different signal types (09_EDS/eds_ana.R)"  />
<p class="caption">
EDS of different signal types (09_EDS/eds_ana.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-pqtl_eds">
Past versions of pqtl_eds.png
</button>
</p>
<div id="fig-pqtl_eds" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/c8cdef503a41584e081ab934af87b44ed87c8374/docs/assets/pqtl_eds.png" target="_blank">c8cdef5</a>
</td>
<td>
liangend
</td>
<td>
2023-09-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="section-2" class="section level2">
<h2>8/11/2023</h2>
<div id="pi1-value-for-significant-pco-signals-in-each-module" class="section level3">
<h3>pi1 value for significant PCO signals in each module</h3>
When using qvalue package to estimate the proportion of null p values, if the number of input p values is too small, estimated pi0 is not available. The mean pi1 is 0.22 across all signals, and the median is 0.16.
<div class="figure" style="text-align: center">
<img src="assets/pco_prot_pi1.png" alt="pi1 values across each module (05_pi1.R)"  />
<p class="caption">
pi1 values across each module (05_pi1.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-pco_prot_pi1">
Past versions of pco_prot_pi1.png
</button>
</p>
<div id="fig-pco_prot_pi1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/69161abcb642bed9ff9dd91ea81f5a8ffc62dfb2/docs/assets/pco_prot_pi1.png" target="_blank">69161ab</a>
</td>
<td>
liangend
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="specificity-of-somamer" class="section level3">
<h3>Specificity of SOMAmer</h3>
<p>Two studies examined the specificity of SOMAmer.</p>
<ol style="list-style-type: decimal">
<li>Emilsson et al. (2018): They confirmed the specificy of SOMAmer using two mass spectrometry techniques: data dependent analysis (DDA) and multiple reaction monitoring (MRM) for 773 aptamers (out of 4137 aptamers corresponding to human protein coding genes). DDA examined 734 and MRM examined 102. Among 63 aptamers examined by both methods, 2 of them disagree with each other.
<div class="figure" style="text-align: center">
<img src="assets/aptamer_spe_ms.png" alt="Aptamer specificity assessed by mass spectrometry (06_aptamer_specificiy.R)"  />
<p class="caption">
Aptamer specificity assessed by mass spectrometry (06_aptamer_specificiy.R)
</p>
</div></li>
</ol>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-aptamer_spe_ms">
Past versions of aptamer_spe_ms.png
</button>
</p>
<div id="fig-aptamer_spe_ms" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/69161abcb642bed9ff9dd91ea81f5a8ffc62dfb2/docs/assets/aptamer_spe_ms.png" target="_blank">69161ab</a>
</td>
<td>
liangend
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="2" style="list-style-type: decimal">
<li>Sun et al. (2018): They tested 920 aptamers (2596 target-related-gene pairs) for detection of proteins with at least 40% sequence homology to the target protein. 126 (14%) aptamers showed comparable binding with a homologous protein, but nearly half (53) of these were binding to alternative forms of the same protein, which result in the rest 73 problematic aptamers (72 of these were also included in Gudjosson et al. (2022)).
<div class="figure" style="text-align: center">
<img src="assets/aptamer_bind_spe.png" alt="Aptamer specificity assessed by binding specificity (06_aptamer_specificiy.R)"  />
<p class="caption">
Aptamer specificity assessed by binding specificity (06_aptamer_specificiy.R)
</p>
</div></li>
</ol>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-aptamer_bind_spe">
Past versions of aptamer_bind_spe.png
</button>
</p>
<div id="fig-aptamer_bind_spe" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/69161abcb642bed9ff9dd91ea81f5a8ffc62dfb2/docs/assets/aptamer_bind_spe.png" target="_blank">69161ab</a>
</td>
<td>
liangend
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="loss-of-function-intolerance-of-pqtl-genes" class="section level3">
<h3>Loss-of-function intolerance of pQTL genes</h3>
<p>There are 3023, 4427, 1403 protein coding genes included in this analysis for Sun et al. (2018), Gudjonsson et al. (2022), and UKB (2022) respectively.</p>
Genes without cis-signals are more likely to be intolerant of loss-of-function variations. Similar results were observed in both datasets.
<div class="figure" style="text-align: center">
<img src="assets/pqtl_pLI.png" alt="pLI of different signal types (08_pqtl_pLI.R)"  />
<p class="caption">
pLI of different signal types (08_pqtl_pLI.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-pqtl_pLI">
Past versions of pqtl_pLI.png
</button>
</p>
<div id="fig-pqtl_pLI" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/69161abcb642bed9ff9dd91ea81f5a8ffc62dfb2/docs/assets/pqtl_pLI.png" target="_blank">69161ab</a>
</td>
<td>
liangend
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="maf-of-pqtl-variants" class="section level3">
<h3>MAF of pQTL variants</h3>
There are in total 1980 pQTLs (589 cis and 1980 trans) from Sun et al. (2018) and 4290 pQTLs (1624 cis and 2666 trans) from Gudjonsson et al. (2022). Cis pQTL leading variants have lower MAF and larger effect size than trans.
<div class="figure" style="text-align: center">
<img src="assets/qptl_maf_beta.png" alt="pQTL MAF and effect size (09_maf_beta.R)"  />
<p class="caption">
pQTL MAF and effect size (09_maf_beta.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-qptl_maf_beta">
Past versions of qptl_maf_beta.png
</button>
</p>
<div id="fig-qptl_maf_beta" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/69161abcb642bed9ff9dd91ea81f5a8ffc62dfb2/docs/assets/qptl_maf_beta.png" target="_blank">69161ab</a>
</td>
<td>
liangend
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
Trans pQTLs overlapping with cis have lower MAF than those don’t.
<div class="figure" style="text-align: center">
<img src="assets/trans_pqtl_maf.png" alt="Trans pQTL MAF (09_maf_beta.R)"  />
<p class="caption">
Trans pQTL MAF (09_maf_beta.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-trans_pqtl_maf">
Past versions of trans_pqtl_maf.png
</button>
</p>
<div id="fig-trans_pqtl_maf" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/69161abcb642bed9ff9dd91ea81f5a8ffc62dfb2/docs/assets/trans_pqtl_maf.png" target="_blank">69161ab</a>
</td>
<td>
liangend
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="chromatin-states-of-pqtl-variants" class="section level3">
<h3>Chromatin states of pQTL variants</h3>
By investigating the chromatin states of lead variants of pQTLs, two studies show different results in blood but consistent results in liver. In both studies, trans-pQTLs are most enriched in ZNF genes &amp; repeats.
<div class="figure" style="text-align: center">
<img src="assets/pqtl_chr_state_blood.png" alt="Blood chromatin states of pQTLs (10_chr_state.R)"  />
<p class="caption">
Blood chromatin states of pQTLs (10_chr_state.R)
</p>
</div>
<div class="figure" style="text-align: center">
<img src="assets/pqtl_chr_state_liver.png" alt="Liver chromatin states of pQTLs (10_chr_state.R)"  />
<p class="caption">
Liver chromatin states of pQTLs (10_chr_state.R)
</p>
</div>
</div>
<div id="colocalization-with-gwas-signals" class="section level3">
<h3>Colocalization with GWAS signals</h3>
Across 35 continuous phenotypes (blood traits, height, weight, LDL, etc.), trans-pQTLs are more likely to be colocalize with GWAS signals (PP4 &gt; 0.5).
<div class="figure" style="text-align: center">
<img src="assets/pqtl_coloc.png" alt="Colocalization summary (11_coloc.R)"  />
<p class="caption">
Colocalization summary (11_coloc.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-pqtl_coloc">
Past versions of pqtl_coloc.png
</button>
</p>
<div id="fig-pqtl_coloc" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/c8cdef503a41584e081ab934af87b44ed87c8374/docs/assets/pqtl_coloc.png" target="_blank">c8cdef5</a>
</td>
<td>
liangend
</td>
<td>
2023-09-06
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/6d243864495892eb8e5c9f341d76d7911e19b833/docs/assets/pqtl_coloc.png" target="_blank">6d24386</a>
</td>
<td>
liangend
</td>
<td>
2023-08-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="section-3" class="section level2">
<h2>7/24/2023</h2>
<div id="protein-qtl-analysis" class="section level3">
<h3>Protein QTL analysis</h3>
Summary stats of AGES cohort from Gudjonsson et al. (2022; Nat Commun). 5368 individuals, 7,506,463 SNPs, 4782 aptamers in total. 27 modules (Emilsson et al., 2018) consisting of 4113 aptamers (corresponding to 3487 protein/protein complex) were analyzed using trans-PCO.
<div class="figure" style="text-align: center">
<img src="assets/prot_mod_div.png" alt="Module division of protein (01_signal_summ.R)"  />
<p class="caption">
Module division of protein (01_signal_summ.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-prot_mod_div">
Past versions of prot_mod_div.png
</button>
</p>
<div id="fig-prot_mod_div" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/69d0acaf048c7a1f152791badd578432be27ae4f/docs/assets/prot_mod_div.png" target="_blank">69d0aca</a>
</td>
<td>
liangend
</td>
<td>
2023-07-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
A total of 84,291 significant (p &lt; 5e-8/27) SNP-module pairs were found in 657 loci (200 kb window around the leading signal) with 39,898 unique SNPs. Most signals were found from module 26, which has molecular functions of signaling receptor binding, peptidase regulator activity, transmembrane receptor protein kinase activity; and involves in biological process of response to stimulus, positive regulation of protein kinase C activity, L-ascorbic acid biosynthetic process. Followed by module 27, which has MF of growth factor binding, signaling receptor binding, cell adhesion molecule binding and BP of anatomical structure development, glycoprotein metabolic process, regulation of protein maturation.
<div class="figure" style="text-align: center">
<img src="assets/prot_signal_dist.png" alt="Protein trans signal distribution (01_signal_summ.R)"  />
<p class="caption">
Protein trans signal distribution (01_signal_summ.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-prot_signal_dist">
Past versions of prot_signal_dist.png
</button>
</p>
<div id="fig-prot_signal_dist" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/69d0acaf048c7a1f152791badd578432be27ae4f/docs/assets/prot_signal_dist.png" target="_blank">69d0aca</a>
</td>
<td>
liangend
</td>
<td>
2023-07-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
The cor between module size and number of associations = 0.52; The cor between module average correlation and number of associations = -0.26; The cor between module average correlation and module size = -0.44
<div class="figure" style="text-align: center">
<img src="assets/mod_size_cor.png" alt="Protein trans signal distribution (01_signal_summ.R)"  />
<p class="caption">
Protein trans signal distribution (01_signal_summ.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-mod_size_cor">
Past versions of mod_size_cor.png
</button>
</p>
<div id="fig-mod_size_cor" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/24d3d2b38d6c69e7484290ba3fed2afed0e79c32/docs/assets/mod_size_cor.png" target="_blank">24d3d2b</a>
</td>
<td>
liangend
</td>
<td>
2023-08-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="comparing-with-univariate-analysis" class="section level3">
<h3>Comparing with univariate analysis</h3>
<p>Using univariate analysis for the 4113 aptamers, there are in total 324,928 significant (p &lt; 5e-8/4728) SNP-protein pairs. 172,608 of them are trans signals (on different chromosomes with the target gene) corresponding to 1403 trans-eGenes and 57729 unique SNPs.</p>
For each protein target, I clustered univariate signals into 200 kb loci around the leading SNP. As a result, 1029 unique loci were found across all genes including 5077 loci-gene pairs. For each loci-gene pair, I found the minimum p value from PCO corresponding to the module containing the target protein. 4581 (90.2%) were replicated in PCO (p &lt; 0.05/5077); 4239 (83.5%) were also significant in PCO (p &lt; 5e-8/27), 3366/5077 (66.3%) has p value &lt; univariate p. 3674/5077 with PCO p &gt; 0 (some p = 0 due to precision issue).
<div class="figure" style="text-align: center">
<img src="assets/univar_pco_signal_comp.png" alt="Signals found by univariate and PCO methods (02_pco_univar_comp.R)"  />
<p class="caption">
Signals found by univariate and PCO methods (02_pco_univar_comp.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-univar_pco_signal_comp">
Past versions of univar_pco_signal_comp.png
</button>
</p>
<div id="fig-univar_pco_signal_comp" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/9ce812645521c9f185681b50e63ebed4c89726ed/docs/assets/univar_pco_signal_comp.png" target="_blank">9ce8126</a>
</td>
<td>
liangend
</td>
<td>
2023-08-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/univar_pco_comp.png" alt="Replication of univariate trans signals using PCO (02_pco_univar_comp.R)"  />
<p class="caption">
Replication of univariate trans signals using PCO (02_pco_univar_comp.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-univar_pco_comp">
Past versions of univar_pco_comp.png
</button>
</p>
<div id="fig-univar_pco_comp" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/9ce812645521c9f185681b50e63ebed4c89726ed/docs/assets/univar_pco_comp.png" target="_blank">9ce8126</a>
</td>
<td>
liangend
</td>
<td>
2023-08-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="assets/minp_comp.png" alt="Minimum p value comparison for 3674 loci (02_pco_univar_comp.R)"  />
<p class="caption">
Minimum p value comparison for 3674 loci (02_pco_univar_comp.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-minp_comp">
Past versions of minp_comp.png
</button>
</p>
<div id="fig-minp_comp" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/6d757dcc70d62eecc4c1455f7c77eff37f5a3612/docs/assets/minp_comp.png" target="_blank">6d757dc</a>
</td>
<td>
liangend
</td>
<td>
2023-08-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>For each protein module in PCO analysis, a similar signal clustering was done to obtain 657 loci including 1595 loci-module pairs.</p>
<div class="figure" style="text-align: center">
<img src="assets/manha_comp.png" alt="Comparison of univariate and PCO P values (04_mahattan.R)"  />
<p class="caption">
Comparison of univariate and PCO P values (04_mahattan.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-manha_comp">
Past versions of manha_comp.png
</button>
</p>
<div id="fig-manha_comp" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/93d1dc418d1d8bb866e9f66709a75d2bec509b01/docs/assets/manha_comp.png" target="_blank">93d1dc4</a>
</td>
<td>
liangend
</td>
<td>
2023-08-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
Then I investigated the number of significant proteins in each unique genome locus, and how this is related to PCO performance.
<div class="figure" style="text-align: center">
<img src="assets/n_prot_per_loc.png" alt="Number of significant proteins per locus in univariate analysis for PCO significant and non-significant loci (02_pco_univar_comp.R)"  />
<p class="caption">
Number of significant proteins per locus in univariate analysis for PCO significant and non-significant loci (02_pco_univar_comp.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-n_prot_per_loc">
Past versions of n_prot_per_loc.png
</button>
</p>
<div id="fig-n_prot_per_loc" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/9ce812645521c9f185681b50e63ebed4c89726ed/docs/assets/n_prot_per_loc.png" target="_blank">9ce8126</a>
</td>
<td>
liangend
</td>
<td>
2023-08-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Number of protein-loci pairs per locus for PCO significant loci:</p>
<p>Mean = 7.3; 25% percentile = 1; median = 2; 75% percentile = 4; max = 984 (# of unique proteins = 571; chr17:26588663-26788663)</p>
<p>Number of significant proteins per locus for PCO non-significant loci:</p>
<p>Mean = 1.4; 25% percentile = 1; median = 1; 75% percentile = 1; max = 9</p>
</div>
<div id="heritability-of-protein" class="section level3">
<h3>Heritability of protein</h3>
For 4782 protein/protein complex, there are 3816 with h2 &gt; 0. Across all proteins, mean h2 = 0.098, median = 0.095, 25% quantile = 0.016, 75% quantile = 0.17, max = 0.81 (gene CTSB, a member of the C1 family of peptidases).
<div class="figure" style="text-align: center">
<img src="assets/h2_dist.png" alt="Protein heritability (03_h2.R)"  />
<p class="caption">
Protein heritability (03_h2.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-h2_dist">
Past versions of h2_dist.png
</button>
</p>
<div id="fig-h2_dist" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/93d1dc418d1d8bb866e9f66709a75d2bec509b01/docs/assets/h2_dist.png" target="_blank">93d1dc4</a>
</td>
<td>
liangend
</td>
<td>
2023-08-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/24d3d2b38d6c69e7484290ba3fed2afed0e79c32/docs/assets/h2_dist.png" target="_blank">24d3d2b</a>
</td>
<td>
liangend
</td>
<td>
2023-08-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
I tried two sets of annotations to estimate h2: 53 baseline annotations (Finucane et al., 2018) and 97 baseline annotations (Hujoel et al., 2019). They agree relatively well with each other (cor = 0.59).
<div class="figure" style="text-align: center">
<img src="assets/h2_cor_annot.png" alt="Cor of h2 between two annotations (03_h2.R)"  />
<p class="caption">
Cor of h2 between two annotations (03_h2.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-h2_cor_annot">
Past versions of h2_cor_annot.png
</button>
</p>
<div id="fig-h2_cor_annot" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/93d1dc418d1d8bb866e9f66709a75d2bec509b01/docs/assets/h2_cor_annot.png" target="_blank">93d1dc4</a>
</td>
<td>
liangend
</td>
<td>
2023-08-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>I calculated the h2 for two datasets (Sun et al., 2018; Gudjonsson et al., 2022). Sun’s data has 3023 protein coding genes, 3301 individuals. They found 1927 significant associations, 34% of which are trans. Gudjonsson’s data has 4427 protein coding genes, 5368 individuals. They found 4035 significant associations, 65% of which are trans.</p>
<p>Define cis as SNPs within the same chromosome as the gene and trans as SNPs in different chromosomes:</p>
<p>For Sun et al., 2018: mean cis h2 = 0.016; mean trans h2 = 0.091; mean trans h2 prop = 0.86</p>
<p>For Gudjonsson et al., 2018: mean cis h2 = 0.043; mean trans h2 = 0.42; mean trans h2 prop = 0.91</p>
<div class="figure" style="text-align: center">
<img src="assets/h2_2_data.png" alt="h2 summary for two datasets (07_h2_2_datasets.R)"  />
<p class="caption">
h2 summary for two datasets (07_h2_2_datasets.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-h2_2_data">
Past versions of h2_2_data.png
</button>
</p>
<div id="fig-h2_2_data" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/69161abcb642bed9ff9dd91ea81f5a8ffc62dfb2/docs/assets/h2_2_data.png" target="_blank">69161ab</a>
</td>
<td>
liangend
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Define cis as SNPs within 1 Mb of the gene tss and trans as SNPs outside 5 Mb of the gene tss and in different chromosomes:</p>
<p>For Sun et al., 2018: mean cis h2 = 0.011; mean trans h2 = 0.096; mean trans h2 prop = 0.90</p>
<p>For Gudjonsson et al., 2018: mean cis h2 = 0.017; mean trans h2 = 0.45; mean trans h2 prop = 0.96</p>
<div class="figure" style="text-align: center">
<img src="assets/h2_2_data_1mb_5mb.png" alt="h2 summary for two datasets (07_h2_2_datasets.R)"  />
<p class="caption">
h2 summary for two datasets (07_h2_2_datasets.R)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-h2_2_data_1mb_5mb">
Past versions of h2_2_data_1mb_5mb.png
</button>
</p>
<div id="fig-h2_2_data_1mb_5mb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liangend/Trans_WIP/blob/c8cdef503a41584e081ab934af87b44ed87c8374/docs/assets/h2_2_data_1mb_5mb.png" target="_blank">c8cdef5</a>
</td>
<td>
liangend
</td>
<td>
2023-09-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: CentOS Linux 8

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el8-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8 LC_NUMERIC=C         LC_TIME=C           
 [4] LC_COLLATE=C         LC_MONETARY=C        LC_MESSAGES=C       
 [7] LC_PAPER=C           LC_NAME=C            LC_ADDRESS=C        
[10] LC_TELEPHONE=C       LC_MEASUREMENT=C     LC_IDENTIFICATION=C 

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.6.2

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.10       rstudioapi_0.13   whisker_0.4       knitr_1.38       
 [5] magrittr_2.0.3    R6_2.5.1          rlang_1.1.1       fansi_1.0.4      
 [9] highr_0.9         stringr_1.5.0     tools_4.1.0       xfun_0.30        
[13] utf8_1.2.3        cli_3.6.1         git2r_0.28.0      jquerylib_0.1.4  
[17] htmltools_0.5.1.1 rprojroot_2.0.2   yaml_2.2.1        digest_0.6.27    
[21] tibble_3.2.1      lifecycle_1.0.3   later_1.2.0       sass_0.4.0       
[25] vctrs_0.6.2       promises_1.2.0.1  fs_1.5.0          glue_1.6.2       
[29] evaluate_0.15     rmarkdown_2.13    stringi_1.7.12    bslib_0.2.5.1    
[33] compiler_4.1.0    pillar_1.9.0      jsonlite_1.7.2    httpuv_1.6.1     
[37] pkgconfig_2.0.3  </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
