---
title: ''
output:
  workflowr::wflow_html:
              toc: true
              toc_depth: 4
editor_options:
  chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=FALSE)
```

```{css style settings, echo = FALSE}
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
```
## 6/21/2023
### 1. Analyzing GTEx using EUR only
Fewer signals were found when decreasing the sample size from all (702) to EUR only (599) for muscle. Similarly for blood (670 vs 574).
```{r echo=F, fig.align='center', fig.cap="Number of significant modules and pairs using all/EUR individuals (16_all_vs_eur_ind.R)"}
knitr::include_graphics(c("assets/eur_vs_all_pco.png"), error = FALSE)
```

For DGN, I randomly subset 574 individuals (equal to GTEx blood EUR sample size), and compared the univariate results. Among all the common genes, DGN dataset had more small p values. For number of eGenes, DGN had 177 and GTEx had 23 (fdr < 0.1).
```{r echo=F, fig.align='center', fig.cap="Number of significant SNP-gene pairs using the same sample size for GTEx blood and DGN (16_all_vs_eur_ind.R)"}
knitr::include_graphics(c("assets/gtex_dgn_univar.png"), error = FALSE)
```

### 2. Geuvadis dataset analysis
Geuvadis dataset contains 358 individuals and 12,382 genes for analysis. The gene expression count was not corrected for mappability. Out of 211 gene modules, there are 159 significant eModules and 13,257 significant SNP-module pairs. The much more signals detected could be due to mappability or data quality.

### 3. Controlling mappability or not for muscle
When not controlling mappability for muscle, I used the normalized TPM from GTEx portal directly to conduct Trans-PCO. To make them comparable, I included the same genes in the analysis. The total number of gene modules is 292 (compared to 301 when controlling mappability).
```{r echo=F, fig.align='center', fig.cap="Control mappability or not for muscle (16_all_vs_eur_ind.R)"}
knitr::include_graphics(c("assets/mappability_muscle.png"), error = FALSE)
```

### 4. Other comparison for DGN

```{r echo=F, fig.align='center', fig.cap="Control mappability or not for muscle (16_all_vs_eur_ind.R)"}
knitr::include_graphics(c("assets/dgn_comp.png"), error = FALSE)
```

### 5. Functional enrichment
```{r echo=F, fig.align='center', fig.cap="Control mappability or not for muscle (06_trans_enrich.R)"}
knitr::include_graphics(c("assets/chrom_enrich_blood.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Control mappability or not for muscle (06_trans_enrich.R)"}
knitr::include_graphics(c("assets/chrom_enrich_muscle.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Control mappability or not for muscle (06_trans_enrich.R)"}
knitr::include_graphics(c("assets/chrom_enrich_thyroid.png"), error = FALSE)
```

### 6. Gene expression correlation
```{r echo=F, fig.align='center', fig.cap="The correlation of gene expression (TPM) for 4 GTEx tissues (17_ex_cor_heat.R)"}
knitr::include_graphics(c("assets/gtex_ex_heat.png"), error = FALSE)
```
Average correlation: 0.844 (muscle), 0.769 (blood), 0.816 (thyroid), 0.875 (testis)

```{r echo=F, fig.align='center', fig.cap="The correlation of gene expression (TPM) for Geuvadis (17_ex_cor_heat.R)"}
knitr::include_graphics(c("assets/geuv_ex_heat.png"), error = FALSE)
```
Average correlation = 0.865


## 5/1/2023

### 1. Distribution of trans signals
32/199 modules were found to have at least one significant trans signal in GTEx blood. In contrast, 102/166 modules were found to have trans-eQTLs in DGN. Most signals were observed in module 135 with 105 SNPs in GTEx (module 51 with 695 SNPs in DGN).
```{r echo=F, fig.align='center', fig.cap="Number of significant SNPs across modules (08_mod_dist.R)"}
knitr::include_graphics(c("assets/mod_dist.png"), error = FALSE)
```

Most signals were found on chromosome 3. In GTEx blood, there are significant 108 SNPs from chr3 (1237 SNPs from chr3 in DGN). Out of these 108 SNPs, 105 SNPs are relative to module 135 (652/1237 SNPs from chr3 are relative to module 51 in DGN).
```{r echo=F, fig.align='center', fig.cap="Number of significant SNPs across chromosomes (08_mod_dist.R)"}
knitr::include_graphics(c("assets/chr_dist.png"), error = FALSE)
```
Genes in module 135 of GTEx: AURKC, HKR1, ZNF134, ZNF135, ZNF211, ZNF256, ZNF304, ZNF329, ZNF347, ZNF418, ZNF543, ZNF544, ZNF548, ZNF549, ZNF550, ZNF551, ZNF585B, ZNF606, ZNF671, ZNF772, ZNF776, ZSCAN18

Genes in module 51 of DGN (61 genes): AURKC, HKR1, LINC00665, PHKB, TRAPPC2P1, ZFP28, ZFP82, ZIK1, ZNF107, ZNF134, ZNF135, ZNF154, ZNF160, ZNF211, ZNF256, ZNF274, ZNF285, ZNF304, ZNF320, ZNF329, ZSCAN18, etc.

20/22 genes in module 135 of GTEx overlaps with module 51 of DGN. Functions of module 135 involves RNA polymerase II transcription regulation and binding, DNA-binding transcription factor activity.

### 2. Replication of GTEx trans signals based on DGN module
There are 256 significant trans SNP-module pairs (p < 5e-8) found in GTEx blood based on DGN modules. 205 of them were also included in the analysis of DGN, and 150 SNP-module pairs were also significant (p < 5e-8)
```{r echo=F, fig.align='center', fig.cap="Trans signal replication between GTEx and DGN"}
knitr::include_graphics(c("assets/gtex_dgn_rep.png"), error = FALSE)
```

### 3. Simulation of the effect of # genes and samples on the detection of significant modules
I simulated a module with 100/200 genes, with one gene causally regulated by a SNP. Heritability of the gene was set to be 0.15. The genotype of the SNP was randomly generated for 10/100/300/500/700/1000/2000 individuals. Expression of other genes were generated from N(0, 1). In total 1000 simulations were conducted for each scenario.

```{r echo=F, fig.align='center', fig.cap="Null p values for gene module with 100 genes (no causal genes were simulated; 07_sim_ana.R)"}
knitr::include_graphics(c("assets/null_p_gene100.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Null p values for gene module with 100 genes (no causal genes were simulated)"}
knitr::include_graphics(c("assets/null_p_hist_gene100.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Null p values for gene module with 200 genes (no causal genes were simulated)"}
knitr::include_graphics(c("assets/null_p_gene200.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Null p values for gene module with 200 genes (no causal genes were simulated)"}
knitr::include_graphics(c("assets/null_p_hist_gene200.png"), error = FALSE)
```

Type I error rate of PCO ACAT under different scenarios
```{r echo=F, fig.align='center', fig.cap="Type I error rate"}
knitr::include_graphics(c("assets/type_I_error_sim.png"), error = FALSE)
```

Power of PCO ACAT under different scenarios
```{r echo=F, fig.align='center', fig.cap="Power analysis"}
knitr::include_graphics(c("assets/power_sim.png"), error = FALSE)
```
The more genes in one module, the more samples required to detect the signal.

### 4. Multiple testing
Use the method in GTEx to find "eModule":

1. Take the most extreme P-value per gene module, and multiply that P-value by 1e6;

2. Then apply Benjamini-Hochberg on the adjusted extreme P-values across gene modules to control FDR.
```{r echo=F, fig.align='center', fig.cap="Number of significant modules detected"}
knitr::include_graphics(c("assets/n_sig_mod.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Number of significant SNP-module pairs detected (99_scripts/08_all_sig.R)"}
knitr::include_graphics(c("assets/n_sig_comp.png"), error = FALSE)
```

Based on permutation FDR correction, there are 108 significant SNP-module pairs from 4 chromosomes and 6 modules, 100 of which were between chr3 and mod135 (nominal p-value threshold = 4.6e-9). 

### 5. Divide large modules into smaller ones
I divided large modules (module size > half of the sample size) into smaller modules (sub module size <= half of the sample size).
```{r echo=F, fig.align='center', fig.cap="Module division comparison (10_mod_div_comp.R)"}
knitr::include_graphics(c("assets/mod_div_comp.png"), error = FALSE)
```

### 6. Summary of 11 tissues
```{r echo=F, fig.align='center', fig.cap="Summary of each tissue (10_sum_11tissues.R)"}
knitr::include_graphics(c("assets/basic_stats.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Significant modules and SNPs (10_sum_11tissues.R)"}
knitr::include_graphics(c("assets/sig_summary.png"), error = FALSE)
```
Most significant pairs were observed from testis, similar as GTEx finding the most trans signals from testis. In total 3361 significant SNP-module pairs found from 28 modules. However, most signals were found between module 168 and chr17 (2855 pairs). Comparing to trans-eGenes found in GTEx (49 genes), 28 modules contains 959 genes, 19 of which overlapped with GTEx ones.
```{r echo=F, fig.align='center', fig.cap="Number of significant SNPs across modules (08_mod_dist.R)"}
knitr::include_graphics(c("assets/snp_dist_testis.png"), error = FALSE)
```

### 7. Using SNPs with MAF > 0.01 for muscle
6,380,580 SNPs with MAF > 0.05; 10,403,248 SNPs with MAF > 0.01.
```{r echo=F, fig.align='center', fig.cap="Module FDR and significant SNP-module pairs (11_maf_comp.R)"}
knitr::include_graphics(c("assets/maf_comp.png"), error = FALSE)
```

### 8. Trans signals overlapping with cis signals
Across all 11 tissues, on average 63.3% of trans signal overlap with cis-eQTLs; 25.2% overlap with cis-sQTLs; 64.9% overlap with either cis-eQTLs or cis-sQTLs.
```{r echo=F, fig.align='center', fig.cap="Overlapping with cis signals (12_overlap_cis.R)"}
knitr::include_graphics(c("assets/overlap_cis.png"), error = FALSE)
```

Across 6 tissues with data of eQTL, sQTL, mQTL and RNA editing, mQTL did not add too much extra information.
```{r echo=F, fig.align='center', fig.cap="Overlapping with cis signals (12_overlap_cis.R)"}
knitr::include_graphics(c("assets/overlap_cis2.png"), error = FALSE)
```

### 9. Colocalization with GWAS signals
Trans-eQTL loci are defined as 100kb flanking regions around leading SNPs.
```{r echo=F, fig.align='center', fig.cap="Colocalization with 4 GWAS traits (13_gwas_coloc.R)"}
knitr::include_graphics(c("assets/gwas_coloc.png"), error = FALSE)
```

Colocalization with 29 blood relevant traits (red blood cell count, platelet count, neutrophil count, etc.) and 3 autoimmune diseases (ulcerative colitis, Crohn's disease and inflammatory bowel disease). Testis contained the most colocalization (27 traits with at least one colocalization), followed by skin (18), blood (15), nerve (11) and thyroid (10). But blood had the highest colocalization proportion (# cololization / # of trans-eQTL loci). "Average" is the average proportion of of coloc across all traits, while "Unique" is the total number of unique loci colocalized with at least one GWAS signal across all traits.
```{r echo=F, fig.align='center', fig.cap="Colocalization with blood and autoimmune disease traits (13_gwas_coloc.R)"}
knitr::include_graphics(c("assets/blood_coloc_prop.png"), error = FALSE)
```

I did the heritability enrichment analysis for one colocalized signal of trans-eQTL (on chr3 associated with module 7) in blood.  Among 32 traits, h2 enrichment of 22 are shown below (p < 0.05). 6 of them reached significance after Bonferroni correction, and 8 of them have a corresponding colocalization signal (PP4 > 0.5).
```{r echo=F, fig.align='center', fig.cap="Heritability enrichment of trans signal on module 7 (14_h2_enrich.R)"}
knitr::include_graphics(c("assets/ldsc_blood_coloc.png"), error = FALSE)
```
For example, I identified an interesting colocalization between MPV (mean platelet volume) associated locus near ARHGEF3 and a trans-eQTL of module 7, which associated with platelet activation and hemostasis (PP4 = 0.68).
```{r echo=F, fig.align='center', fig.cap="Colocalization between MPV GWAS and trans signal on module 7 (14_h2_enrich.R)"}
knitr::include_graphics(c("assets/coloc_MPV_blood.png"), error = FALSE)
```

### 10. Model calibration
Given the limited number of significant SNPs I observe, I tried to include different number of covariates in the model to see which gives the most significant signals. I tried 4 different scenarios on muscle: 1) 60 expression PCs + 5 genotype PCs + 3 covariates; 2) 10 expression PCs + 5 genotype PCs + 3 covariates; 3) 5 genotype PCs + 3 covariates; 4) No covariates.

```{r echo=F, fig.align='center', fig.cap="QQ plot of p values for chr22 (15_pval_calib.R)"}
knitr::include_graphics(c("assets/qq_obs_muscle_chr22.png"), error = FALSE)
```

Number of significant eModules and eSNPs:

1) 68 covaraites: 33 eModules, 417 eSNPs;

2) 18 covaraites: 10 eModules, 180 eSNPs;

3) 8 covaraites: 3 eModules, 80 eSNPs;

4) No covaraites: 274 eModules, 2,430,382 eSNPs.

Null p distribution
```{r echo=F, fig.align='center', fig.cap="QQ plot of p values for chr20 (15_pval_calib.R)"}
knitr::include_graphics(c("assets/qq_muscle_chr20.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Histogram of p values for chr20 (15_pval_calib.R)"}
knitr::include_graphics(c("assets/histP_muscle_chr20.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="QQ plot of p values for chr21 (15_pval_calib.R)"}
knitr::include_graphics(c("assets/qq_muscle_chr21.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Histogram of p values for chr21 (15_pval_calib.R)"}
knitr::include_graphics(c("assets/histP_muscle_chr21.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="QQ plot of p values for chr22 (15_pval_calib.R)"}
knitr::include_graphics(c("assets/qq_muscle_chr22.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Histogram of p values for chr22 (15_pval_calib.R)"}
knitr::include_graphics(c("assets/histP_muscle_chr22.png"), error = FALSE)
```


## 4/12/2023

### 1. New read count results based on adjusted GENCODE gtf (for GTEx V8)
After QC and removing genes on chrX, chrM and chrY, 12929 genes for 670 individuals were kept. Among these genes, 12039 are protein coding genes and lincRNA.

#### Compare my TPM with GTEx TPM (including genes on chrX, chrM and chrY)
14047 genes with cor > 0.85; 32 genes with cor <= 0.85
```{r echo=F, fig.align='center', fig.cap="Cor between GTEx TPM and my TPM"}
knitr::include_graphics(c("assets/tpm_cor.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="TPM of selected genes"}
knitr::include_graphics(c("assets/gene_tpm.png"), error = FALSE)
```

13196 genes with mean TPM < 100 for both groups.
```{r echo=F, fig.align='center', fig.cap="Mean TPM across 670 individuals"}
knitr::include_graphics(c("assets/mean_tpm.png"), error = FALSE)
```

### 2. Compare PC with PEER
For the normalized TPM, the first 60 PCs and PEER factors are highly correlated.
```{r echo=F, fig.align='center', fig.cap="Correlation between PC and PEER factor"}
knitr::include_graphics(c("assets/pc_peer_cor.png"), error = FALSE)
```
Zhou et al. (2022) suggested that PCA performs close to or better than PEER factors for QTL mapping under different scenarios, but with much shorter time and better interpretability.

### 3. Coexpression analysis
I used 5 genotype PCs, 3 factors (sex, pcr, platform), and different numbers (10 and 60) of expression PCs or PEER factors as the covariates to obtain the residual expression, which is further used as the input of WGCNA.

I used the elbow method (Zhou et al., 2022) the optimized the number of PCs to be included. 
```{r echo=F, fig.align='center', fig.cap="Optimize the number of PCs using the elbow method"}
knitr::include_graphics(c("assets/pc_elbow.png"), error = FALSE)
```
Another optimization method, BE algorithm, tends to include more PCs than the elbow method, but both are fewer the number of PEER factors included in the GTEx paper.

Number of modules, unassigned genes, and genes in top 3 modules when including different number of PCs or PEER factors.
```{r echo=F, fig.align='center', fig.cap=""}
knitr::include_graphics(c("assets/coexp_comp.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Cluster dendrogram of genes when including 19 PCs"}
knitr::include_graphics(c("assets/wgcna_dendro_19pc.png"), error = FALSE)
```

For trans-PCO analysis, I conducted co-expression analysis based on 5 genotype PCs, 3 factors (sex, pcr, platform), and 10 expression PCs, which resulted in 199 modules, 235 unassigned genes, and top 3 modules containing 858, 481, and 475 genes.

#### Functional profiling of gene modules
To check the sanity of gene module classification, I investigated the enrichment of GO terms (FDR < 0.05) for several gene modules using g:Profiler.
```{r echo=F, fig.align='center', fig.cap="Shared and unique GO terms for module 5, 6 and 7"}
knitr::include_graphics(c("assets/mod_go.png"), error = FALSE)
```
Top 3 significant BP terms:

module 5 (313 genes): leukocyte activation, cell activation, lymphocyte activation;

module 6 (312 genes): defense response to symbiont, defense response to virus, response to virus;

module 7 (307 genes): wound healing, platelet activation, blood coagulation.

Another example:
```{r echo=F, fig.align='center', fig.cap="Shared and unique GO terms for module 106, 107 and 108"}
knitr::include_graphics(c("assets/mod_go2.png"), error = FALSE)
```
Top 3 significant BP terms:

module 106 (31 genes): extracellular structure organization, extracellular matrix organization, external encapsulating structure organization;

module 107 (31 genes): monosaccharide metabolic process, hexose metabolic process, glucose metabolic process;

module 108 (31 genes): None

### 4. cis-eQTL analysis
Included 5 genotype PCs, 3 factors (sex, pcr, platform), 10 expression PCs as covariates. 

Genotype data: 46,569,704 variants after QC. 10,726,114 SNPs (MAF >= 0.01) were included in the cis-eQTL analysis.

12360 eGenes (qval < 0.05) in GTEx data; 6067 eGenes in my analysis. Among 12360 eGenes reported by GTEx, there are 3858 genes not included in our analysis.
```{r echo=F, fig.align='center', fig.cap="Comparison of number of eGenes"}
knitr::include_graphics(c("assets/egene_venn.png"), error = FALSE)
```

1906 same top variants associated with the same eGene. Slope cor = 0.99
```{r echo=F, fig.align='center', fig.cap="Comparison of the slopes of shared variants"}
knitr::include_graphics(c("assets/cis_eqtl_slope_comp.png"), error = FALSE)
```

When using the same covarites (60 PEER factors) as GTEx, the number of eGenes increases to 8320.

### 5. trans-eQTL analysis
Included 5 genotype PCs, 3 factors (sex, pcr, platform), 60 expression PCs as covariates. 

Genotype data: 6,380,580 SNPs (MAF >= 0.05; 6.2 million SNPs in DGN) were included in the trans-eQTL analysis.

#### trans-eQTL mapping based on 166 gene modules from Wang et al. (2022)
In trans-eQTL mapping for DGN, there are 3899 significant trans-eQTL SNP–module pairs (Wang et al., 2022). For the 12132 genes (166 gene modules) included in Wang et al. (2022), there are 10547 genes included in my analysis. Among 3330 shared SNP-gene module pairs, 470 (14.0%) were replicated in my analysis (p < 0.05/3330).
```{r echo=F, fig.align='center', fig.cap="Trans-eQTL replication for DGN"}
knitr::include_graphics(c("assets/trans_sig_rep.png"), error = FALSE)
```

#### trans-eQTL mapping based on 199 gene modules
In total 159 significant signals were found in GTEx data (p < 5e-8). I then investigated the regulatory activity of these signals and those from Wang et al. (2022) based on chromatin states of neutrophils from peripheral blood (Roadmap project).
```{r echo=F, fig.align='center', fig.cap="Functional enrichment of significant trans signals"}
knitr::include_graphics(c("assets/trans_func_enrich.png"), error = FALSE)
```

GTEx V8 reported 13 trans-eQTLs in whole blood (Wang et al., 2022), and 4 (30.8%) were replicated in my analysis (p < 0.05/13).
```{r echo=F, fig.align='center', fig.cap="Trans-eQTL replication for GTEx"}
knitr::include_graphics(c("assets/trans_gtex_rep.png"), error = FALSE)
```

## 3/8/2023

### 1. Comparing the p values of PCO and ACAT on Lili's demo
Number of variant-module associations = 136,904. Correlation of two p values = 0.77

Correlation of two p values (both < 0.05) = 0.73
```{r echo=F, fig.align='center', fig.cap="Comparing the p values of PCO and ACAT"}
knitr::include_graphics(c("assets/pco_acat_comp.png"), error = FALSE)
```

## 2/21/2023

### 1. Summary

Use GTEx data to generate a whole-genome Trans-eQTL map based on TransPCO.

### 2. Use whole blood data as the initial test

- 670 samples of RNAseq

- 26584 genes

- Filter by mapping quality = 255 (uniquely mapping reads; [STAR mapping manual](https://physiology.med.cornell.edu/faculty/skrabanek/lab/angsd/lecture_notes/STARmanual.pdf))

- More strict way to count read
![](assets/count_modes.png){width="90%"}
- Filter by CPM (counts per million)

For each gene, count the number of samples with CPM > 0.5. 11771 genes kept if filter by number of samples with CPM > 0.5 greater than 10% of total samples.
```{r echo=F, fig.align='center', fig.cap="Figure 1: Number of genes kept with different CPM filtering criteria."}
knitr::include_graphics(c("assets/Fig1.png"), error = FALSE)
```

- Normalize RPKM (Reads per kilo base of transcript per million)
```{r out.width="80%", fig.cap="Figure 2: Density of normalized RPKM of randomly selected 10 genes.", fig.align='center', echo=F}
knitr::include_graphics(c("assets/Fig2.png"), error = FALSE)
```

The problematic left tails are caused by too many ties (0's) for genes that are only found in few samples.
```{r out.width="80%", fig.cap="Figure 3: Density of normalized RPKM and original counts of 3 example genes.",fig.align='center', echo=F}
knitr::include_graphics(c("assets/Fig3.png"), error = FALSE)
```


When filtering by number of samples with CPM > 0.5 greater than 20% of total samples (10003 genes kept).
```{r out.width="80%", fig.cap="Figure 4: Density of normalized RPKM of randomly selected 10 genes using 20% of samples as cutoff.",fig.align='center', echo=F}
knitr::include_graphics(c("assets/Fig4.png"), error = FALSE)
```



