---
title: ''
output:
  workflowr::wflow_html:
              toc: true
              toc_depth: 4
editor_options:
  chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=FALSE)
```

```{css style settings, echo = FALSE}
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
```

## 2/21/2023

### 1. Summary

Use GTEx data to generate a whole-genome Trans-eQTL map based on TransPCO.

### 2. Use whole blood data as the initial test

- 670 samples of RNAseq

- 26584 genes

- Filter by mapping quality = 255 (uniquely mapping reads; [STAR mapping manual](https://physiology.med.cornell.edu/faculty/skrabanek/lab/angsd/lecture_notes/STARmanual.pdf))

- More strict way to count read
![](assets/count_modes.png){width="90%"}
- Filter by CPM (counts per million)

For each gene, count the number of samples with CPM > 0.5. 11771 genes kept if filter by number of samples with CPM > 0.5 greater than 10% of total samples.
```{r echo=F, fig.align='center', fig.cap="Figure 1: Number of genes kept with different CPM filtering criteria."}
knitr::include_graphics(c("assets/Fig1.png"), error = FALSE)
```

- Normalize RPKM (Reads per kilo base of transcript per million)
```{r out.width="80%", fig.cap="Figure 2: Density of normalized RPKM of randomly selected 10 genes.", fig.align='center', echo=F}
knitr::include_graphics(c("assets/Fig2.png"), error = FALSE)
```

The problematic left tails are caused by too many ties (0's) for genes that are only found in few samples.
```{r out.width="80%", fig.cap="Figure 3: Density of normalized RPKM and original counts of 3 example genes.",fig.align='center', echo=F}
knitr::include_graphics(c("assets/Fig3.png"), error = FALSE)
```


When filtering by number of samples with CPM > 0.5 greater than 20% of total samples (10003 genes kept).
```{r out.width="80%", fig.cap="Figure 4: Density of normalized RPKM of randomly selected 10 genes using 20% of samples as cutoff.",fig.align='center', echo=F}
knitr::include_graphics(c("assets/Fig4.png"), error = FALSE)
```

## Another week {#header-of-another-section}

### 1. Analysis 1

#### Analysis

#### Figures, tables

#### Observations



### 2. Analysis 2

