---
title: ''
output:
  workflowr::wflow_html:
              toc: true
              toc_depth: 4
editor_options:
  chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=FALSE)
```

```{css style settings, echo = FALSE}
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
```
## 3/24/2023

### 1. New read count results based on adjusted GENCODE gtf (for GTEx V8)
After QC and removing genes on chrX, chrM and chrY, 12929 genes for 670 individuals were kept. Among these genes, 12039 are protein coding genes and lincRNA.

#### Compare my TPM with GTEx TPM (including genes on chrX, chrM and chrY)
14047 genes with cor > 0.85; 32 genes with cor <= 0.85
```{r echo=F, fig.align='center', fig.cap="Cor between GTEx TPM and my TPM"}
knitr::include_graphics(c("assets/tpm_cor.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="TPM of selected genes"}
knitr::include_graphics(c("assets/gene_tpm.png"), error = FALSE)
```

13196 genes with mean TPM < 100 for both groups.
```{r echo=F, fig.align='center', fig.cap="Mean TPM across 670 individuals"}
knitr::include_graphics(c("assets/mean_tpm.png"), error = FALSE)
```

### 2. Compare PC with PEER
For the normalized TPM, the first 60 PCs and PEER factors are highly correlated.
```{r echo=F, fig.align='center', fig.cap="Correlation between PC and PEER factor"}
knitr::include_graphics(c("assets/pc_peer_cor.png"), error = FALSE)
```
Zhou et al. (2022) suggested that PCA performs close to or better than PEER factors for QTL mapping under different scenarios, but with much shorter time and better interpretability.

### 3. Coexpression analysis
I used 5 genotype PCs, 3 factors (sex, pcr, platform), and different numbers (10 and 60) of expression PCs or PEER factors as the covariates to obtain the residual expression, which is further used as the input of WGCNA.

I used the elbow method (Zhou et al., 2022) the optimized the number of PCs to be included. 
```{r echo=F, fig.align='center', fig.cap="Optimize the number of PCs using the elbow method"}
knitr::include_graphics(c("assets/pc_elbow.png"), error = FALSE)
```
Another optimization method, BE algorithm, tends to include more PCs than the elbow method, but both are fewer the number of PEER factors included in the GTEx paper.

Number of modules, unassigned genes, and genes in top 3 modules when including different number of PCs or PEER factors.
```{r echo=F, fig.align='center', fig.cap=""}
knitr::include_graphics(c("assets/coexp_comp.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Cluster dendrogram of genes when including 19 PCs"}
knitr::include_graphics(c("assets/wgcna_dendro_19pc.png"), error = FALSE)
```

For trans-PCO analysis, I conducted co-expression analysis based on 5 genotype PCs, 3 factors (sex, pcr, platform), and 10 expression PCs, which resulted in 199 modules, 235 unassigned genes, and top 3 modules containing 858, 481, and 475 genes.


### 4. cis-eQTL analysis
Included 5 genotype PCs, 3 factors (sex, pcr, platform), 10 expression PCs as covariates. 

Genotype data: 46,569,704 variants after QC. 10,726,114 SNPs (MAF >= 0.01) were included in the cis-eQTL analysis.

12360 eGenes (qval < 0.05) in GTEx data; 6067 eGenes in my analysis. Among 12360 eGenes reported by GTEx, there are 3858 genes not included in our analysis.
```{r echo=F, fig.align='center', fig.cap="Comparison of number of eGenes"}
knitr::include_graphics(c("assets/egene_venn.png"), error = FALSE)
```

1906 same top variants associated with the same eGene. Slope cor = 0.99
```{r echo=F, fig.align='center', fig.cap="Comparison of the slopes of shared variants"}
knitr::include_graphics(c("assets/cis_eqtl_slope_comp.png"), error = FALSE)
```

When using the same covarites (60 PEER factors) as GTEx, the number of eGenes increases to 8320.

### 5. trans-eQTL analysis
Included 5 genotype PCs, 3 factors (sex, pcr, platform), 60 expression PCs as covariates. 

Genotype data: 6,568,202 SNPs (MAF >= 0.05; 6.2 million SNPs in DGN) were included in the trans-eQTL analysis.

In trans-eQTL mapping for DGN, there are 3899 trans-eQTL SNPâ€“module pairs (Wang et al., 2022). 3421 of the SNPs were also included in the GTEx genotype. For 98 signals found from chr20-22, 5 (5.1%) were replicated in my analysis (p < 0.05/3421).

## 3/8/2023

### 1. Comparing the p values of PCO and ACAT on Lili's demo
Number of variant-module associations = 136,904. Correlation of two p values = 0.77

Correlation of two p values (both < 0.05) = 0.73
```{r echo=F, fig.align='center', fig.cap="Comparing the p values of PCO and ACAT"}
knitr::include_graphics(c("assets/pco_acat_comp.png"), error = FALSE)
```

## 2/21/2023

### 1. Summary

Use GTEx data to generate a whole-genome Trans-eQTL map based on TransPCO.

### 2. Use whole blood data as the initial test

- 670 samples of RNAseq

- 26584 genes

- Filter by mapping quality = 255 (uniquely mapping reads; [STAR mapping manual](https://physiology.med.cornell.edu/faculty/skrabanek/lab/angsd/lecture_notes/STARmanual.pdf))

- More strict way to count read
![](assets/count_modes.png){width="90%"}
- Filter by CPM (counts per million)

For each gene, count the number of samples with CPM > 0.5. 11771 genes kept if filter by number of samples with CPM > 0.5 greater than 10% of total samples.
```{r echo=F, fig.align='center', fig.cap="Figure 1: Number of genes kept with different CPM filtering criteria."}
knitr::include_graphics(c("assets/Fig1.png"), error = FALSE)
```

- Normalize RPKM (Reads per kilo base of transcript per million)
```{r out.width="80%", fig.cap="Figure 2: Density of normalized RPKM of randomly selected 10 genes.", fig.align='center', echo=F}
knitr::include_graphics(c("assets/Fig2.png"), error = FALSE)
```

The problematic left tails are caused by too many ties (0's) for genes that are only found in few samples.
```{r out.width="80%", fig.cap="Figure 3: Density of normalized RPKM and original counts of 3 example genes.",fig.align='center', echo=F}
knitr::include_graphics(c("assets/Fig3.png"), error = FALSE)
```


When filtering by number of samples with CPM > 0.5 greater than 20% of total samples (10003 genes kept).
```{r out.width="80%", fig.cap="Figure 4: Density of normalized RPKM of randomly selected 10 genes using 20% of samples as cutoff.",fig.align='center', echo=F}
knitr::include_graphics(c("assets/Fig4.png"), error = FALSE)
```



