---
title: ''
output:
  workflowr::wflow_html:
              toc: true
              toc_depth: 4
editor_options:
  chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=FALSE)
```

```{css style settings, echo = FALSE}
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
```
## 7/24/2023
### Protein QTL analysis
Summary stats of AGES cohort from Gudjonsson et al. (2022; Nat Commun). 5368 individuals, 7,506,463 SNPs, 4782 aptamers in total. 27 modules (Emilsson et al., 2018) consisting of 4113 aptamers (corresponding to 3487 protein/protein complex) were analyzed using trans-PCO. 
```{r echo=F, fig.align='center', fig.cap="Module division of protein (01_signal_summ.R)"}
knitr::include_graphics(c("assets/prot_mod_div.png"), error = FALSE)
```

A total of 84,291 significant (p < 5e-8/27) SNP-module pairs were found in 657 loci (200 kb window around the leading signal) with 39,898 unique SNPs. Most signals were found from module 26, which has molecular functions of signaling receptor binding, peptidase regulator activity, transmembrane receptor protein kinase activity; and involves in biological process of response to stimulus, positive regulation of protein kinase C activity, L-ascorbic acid biosynthetic process. Followed by module 27, which has MF of growth factor binding, signaling receptor binding, cell adhesion molecule binding and BP of anatomical structure development, glycoprotein metabolic process, regulation of protein maturation.
```{r echo=F, fig.align='center', fig.cap="Protein trans signal distribution (01_signal_summ.R)"}
knitr::include_graphics(c("assets/prot_signal_dist.png"), error = FALSE)
```

The cor between module size and number of associations = 0.52;
The cor between module average correlation and number of associations = -0.26;
The cor between module average correlation and module size  = -0.44
```{r echo=F, fig.align='center', fig.cap="Protein trans signal distribution (01_signal_summ.R)"}
knitr::include_graphics(c("assets/mod_size_cor.png"), error = FALSE)
```

### Comparing with univariate analysis
Using univariate analysis for the 4113 aptamers, there are in total 324,928 significant (p < 5e-8/4728) SNP-protein pairs. 172,608 of them are trans signals (on different chromosomes with the target gene) corresponding to 1403 trans-eGenes and 57729 unique SNPs.

For each protein target, I clustered univariate signals into 200 kb loci around the leading SNP. As a result, 1029 unique loci were found across all genes including 5077 loci-gene pairs. For each loci-gene pair, I found the minimum p value from PCO corresponding to the module containing the target protein. 4581 (90.2%) were replicated in PCO (p < 0.05/5077); 4239 (83.5%) were also significant in PCO (p < 5e-8/27), 3366/5077 (66.3%) has p value < univariate p. 3674/5077 with PCO p > 0 (some p = 0 due to precision issue).
```{r echo=F, fig.align='center', fig.cap="Signals found by univariate and PCO methods (02_pco_univar_comp.R)"}
knitr::include_graphics(c("assets/univar_pco_signal_comp.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Replication of univariate trans signals using PCO (02_pco_univar_comp.R)"}
knitr::include_graphics(c("assets/univar_pco_comp.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Minimum p value comparison for 3674 loci (02_pco_univar_comp.R)"}
knitr::include_graphics(c("assets/minp_comp.png"), error = FALSE)
```

For each protein module in PCO analysis, a similar signal clustering was done to obtain 657 loci including 1595 loci-module pairs.

```{r echo=F, fig.align='center', fig.cap="Comparison of univariate and PCO P values (04_mahattan.R)"}
knitr::include_graphics(c("assets/manha_comp.png"), error = FALSE)
```

Then I investigated the number of significant proteins in each unique genome locus, and how this is related to PCO performance.
```{r echo=F, fig.align='center', fig.cap="Number of significant proteins per locus in univariate analysis for PCO significant and non-significant loci (02_pco_univar_comp.R)"}
knitr::include_graphics(c("assets/n_prot_per_loc.png"), error = FALSE)
```
Number of protein-loci pairs per locus for PCO significant loci: 

Mean = 7.3; 25% percentile = 1; median = 2; 75% percentile = 4; max = 984 (# of unique proteins = 571; chr17:26588663-26788663)

Number of significant proteins per locus for PCO non-significant loci: 

Mean = 1.4; 25% percentile = 1; median = 1; 75% percentile = 1; max = 9

### Heritability of protein
For 4782 protein/protein complex, there are 3816 with h2 > 0. Across all proteins, mean h2 = 0.098, median = 0.095, 25% quantile = 0.016, 75% quantile = 0.17, max = 0.81 (gene CTSB, a member of the C1 family of peptidases).
```{r echo=F, fig.align='center', fig.cap="Protein heritability (03_h2.R)"}
knitr::include_graphics(c("assets/h2_dist.png"), error = FALSE)
```
I tried two sets of annotations to estimate h2: 53 baseline annotations (Finucane et al., 2018) and 97 baseline annotations (Hujoel et al., 2019). They agree relatively well with each other (cor = 0.59).
```{r echo=F, fig.align='center', fig.cap="Cor of h2 between two annotations (03_h2.R)"}
knitr::include_graphics(c("assets/h2_cor_annot.png"), error = FALSE)
```

Using method from Liu et al. (2017), mean h2 = 0.070, 91.9% of which were explained by trans effect (inter-chr), while 9.2% were explained by cis effect (intra-chr).



