---
title: ''
output:
  workflowr::wflow_html:
              toc: true
              toc_depth: 4
editor_options:
  chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=FALSE)
```

```{css style settings, echo = FALSE}
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
```
## 9/25/2023
### Trans-signal of protein complex using PCO
UKB protein summary stats have 2940 summary stats from 33995 individuals. 2837 of summary stats target on 2818 genes on autosomes without multiple targets. 

I used a protein complex database, CORUM, to define protein modules PCO analysis of UKB data. There are in total 3637 human protein complexes in CORUM database, including 15033 proteins (4421 unique ones). 598 protein complexes overlap with > 1 protein in UKB data, including 1436 proteins (535 unique ones). The number of proteins in these complexes varies from 2 to 13 (mean: 2.4; median: 2). 

There are 966,157 significant module-SNP pairs (p < 5e-8/598). By grouping them into 200 kb loci around leading SNPs, there are 12155 module-loci pairs. By comparing to minimum p in the loci for each protein in UKB univariate analysis, there are 1676 loci only significant in PCO, 4957 loci with smaller p values in PCO.
```{r echo=F, fig.align='center', fig.cap="Comparison of univariate and PCO results for UKB data (14_ukb_pco_mahattan.R)"}
knitr::include_graphics(c("assets/ukb_pco_univar_comp.png"), error = FALSE)
```

Among 12155 trans-pQTLs, 1465 of them are located on the same chromosome of one of proteins in the targeting protein complex. Average distance of leading SNP to the nearest gene is 35,964,293, median distance is 20,441,333. 265 of them are within 1 Mb.
```{r echo=F, fig.align='center', fig.cap="Comparison of univariate and PCO results for UKB data (15_ukb_pco_summ.R)"}
knitr::include_graphics(c("assets/pqtl_gene_dist.png"), error = FALSE)
```

### Colocalization with GWAS signals
Across 35 continuous phenotypes (blood traits, height, weight, LDL, etc.), trans-PCO signals are more likely to colocalize with GWAS signals (PP4 > 0.5).
```{r echo=F, fig.align='center', fig.cap="Colocalization summary (11_coloc.R)"}
knitr::include_graphics(c("assets/pco_pqtl_coloc.png"), error = FALSE)
```

## 9/13/2023
### Enhancer length of pQTL genes
I first use enhancer annotation in liver. Genes with cis signals have longer enhancer regions then those without cis signals. Error bars were generated based on bootstrapping for 1000 samples. 
```{r echo=F, fig.align='center', fig.cap="Enhancer length for pQTL genes (12_enh_length.R)"}
knitr::include_graphics(c("assets/enh_length.png"), error = FALSE)
```
When using the average enhancer length across all tissues and the number of active tissues (containing at least one enhancer) as the covariates to do logistic regression (Mostafavi et al., 2022), enhancer length does not have a significant effect on the gene having a cis signal or not.
```{r echo=F, fig.align='center', fig.cap="Enhancer length for pQTL genes (12_enh_length.R)"}
knitr::include_graphics(c("assets/enh_len_logit_reg.png"), error = FALSE)
```
Gene length is positively correlated with pLI (cor = 0.16); average enhancer length is also positively correlated with pLI (0.15) and gene length (0.08).
```{r echo=F, fig.align='center', fig.cap="Enhancer length, gene length and pLI for genes with at least one enhancer annotation (12_enh_length.R)"}
knitr::include_graphics(c("assets/pLI_enh_gene_len.png"), error = FALSE)
```

### Effect size of most significant cis and trans signals
Unlike eQTL, the most extreme cis and trans effects (smallest p value) in protein are not significantly different.
```{r echo=F, fig.align='center', fig.cap="Extreme cis and trans effect size for each protein (13_most_sig_cis_trans.R)"}
knitr::include_graphics(c("assets/most_sig_beta.png"), error = FALSE)
```

## 9/6/2023
### Enhancer domain score (EDS) of pQTL genes
No EDS differences were found between cis-pQTL and trans-pQTL genes
```{r echo=F, fig.align='center', fig.cap="EDS of different signal types (09_EDS/eds_ana.R)"}
knitr::include_graphics(c("assets/pqtl_eds.png"), error = FALSE)
```

## 8/11/2023
### pi1 value for significant PCO signals in each module
When using qvalue package to estimate the proportion of null p values, if the number of input p values is too small, estimated pi0 is not available. The mean pi1 is 0.22 across all signals, and the median is 0.16.
```{r echo=F, fig.align='center', fig.cap="pi1 values across each module (05_pi1.R)"}
knitr::include_graphics(c("assets/pco_prot_pi1.png"), error = FALSE)
```

### Specificity of SOMAmer
Two studies examined the specificity of SOMAmer.

1. Emilsson et al. (2018): They confirmed the specificy of SOMAmer using two mass spectrometry techniques: data dependent analysis (DDA) and multiple reaction monitoring (MRM) for 773 aptamers (out of 4137 aptamers corresponding to human protein coding genes). DDA examined 734 and MRM examined 102. Among 63 aptamers examined by both methods, 2 of them disagree with each other.
```{r echo=F, fig.align='center', fig.cap="Aptamer specificity assessed by mass spectrometry (06_aptamer_specificiy.R)"}
knitr::include_graphics(c("assets/aptamer_spe_ms.png"), error = FALSE)
```

2. Sun et al. (2018): They tested 920 aptamers (2596 target-related-gene pairs) for detection of proteins with at least 40% sequence homology to the target protein. 126 (14%) aptamers showed comparable binding with a homologous protein, but nearly half (53) of these were binding to alternative forms of the same protein, which result in the rest 73 problematic aptamers (72 of these were also included in Gudjosson et al. (2022)).
```{r echo=F, fig.align='center', fig.cap="Aptamer specificity assessed by binding specificity (06_aptamer_specificiy.R)"}
knitr::include_graphics(c("assets/aptamer_bind_spe.png"), error = FALSE)
```

### Loss-of-function intolerance of pQTL genes
There are 3023, 4427, 1403 protein coding genes included in this analysis for Sun et al. (2018), Gudjonsson et al. (2022), and UKB (2022) respectively.

Genes without cis-signals are more likely to be intolerant of loss-of-function variations. Similar results were observed in both datasets.
```{r echo=F, fig.align='center', fig.cap="pLI of different signal types (08_pqtl_pLI.R)"}
knitr::include_graphics(c("assets/pqtl_pLI.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="pLI against |Z| quantiles (08_pqtl_pLI.R)"}
knitr::include_graphics(c("assets/pqtl_z_pLI.png"), error = FALSE)
```

### MAF of pQTL variants
There are in total 1980 pQTLs (589 cis and 1980 trans) from Sun et al. (2018) and 4290 pQTLs (1624 cis and 2666 trans) from Gudjonsson et al. (2022). Cis pQTL leading variants have lower MAF and larger effect size than trans.
```{r echo=F, fig.align='center', fig.cap="pQTL MAF and effect size (09_maf_beta.R)"}
knitr::include_graphics(c("assets/qptl_maf_beta.png"), error = FALSE)
```

Trans pQTLs overlapping with cis have lower MAF than those don't.
```{r echo=F, fig.align='center', fig.cap="Trans pQTL MAF (09_maf_beta.R)"}
knitr::include_graphics(c("assets/trans_pqtl_maf.png"), error = FALSE)
```

### Chromatin states of pQTL variants
By investigating the chromatin states of lead variants of pQTLs, two studies show different results in blood but consistent results in liver. In both studies, trans-pQTLs are most enriched in ZNF genes & repeats.
```{r echo=F, fig.align='center', fig.cap="Blood chromatin states of pQTLs (10_chr_state.R)"}
knitr::include_graphics(c("assets/pqtl_chr_state_blood.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Liver chromatin states of pQTLs (10_chr_state.R)"}
knitr::include_graphics(c("assets/pqtl_chr_state_liver.png"), error = FALSE)
```

### Colocalization with GWAS signals
Across 35 continuous phenotypes (blood traits, height, weight, LDL, etc.), trans-pQTLs are more likely to be colocalize with GWAS signals (PP4 > 0.5).
```{r echo=F, fig.align='center', fig.cap="Colocalization summary (11_coloc.R)"}
knitr::include_graphics(c("assets/pqtl_coloc.png"), error = FALSE)
```


## 7/24/2023
### Protein QTL analysis
Summary stats of AGES cohort from Gudjonsson et al. (2022; Nat Commun). 5368 individuals, 7,506,463 SNPs, 4782 aptamers in total. 27 modules (Emilsson et al., 2018) consisting of 4113 aptamers (corresponding to 3487 protein/protein complex) were analyzed using trans-PCO. 
```{r echo=F, fig.align='center', fig.cap="Module division of protein (01_signal_summ.R)"}
knitr::include_graphics(c("assets/prot_mod_div.png"), error = FALSE)
```

A total of 84,291 significant (p < 5e-8/27) SNP-module pairs were found in 657 loci (200 kb window around the leading signal) with 39,898 unique SNPs. Most signals were found from module 26, which has molecular functions of signaling receptor binding, peptidase regulator activity, transmembrane receptor protein kinase activity; and involves in biological process of response to stimulus, positive regulation of protein kinase C activity, L-ascorbic acid biosynthetic process. Followed by module 27, which has MF of growth factor binding, signaling receptor binding, cell adhesion molecule binding and BP of anatomical structure development, glycoprotein metabolic process, regulation of protein maturation.
```{r echo=F, fig.align='center', fig.cap="Protein trans signal distribution (01_signal_summ.R)"}
knitr::include_graphics(c("assets/prot_signal_dist.png"), error = FALSE)
```

The cor between module size and number of associations = 0.52;
The cor between module average correlation and number of associations = -0.26;
The cor between module average correlation and module size  = -0.44
```{r echo=F, fig.align='center', fig.cap="Protein trans signal distribution (01_signal_summ.R)"}
knitr::include_graphics(c("assets/mod_size_cor.png"), error = FALSE)
```

### Comparing with univariate analysis
Using univariate analysis for the 4113 aptamers, there are in total 324,928 significant (p < 5e-8/4728) SNP-protein pairs. 172,608 of them are trans signals (on different chromosomes with the target gene) corresponding to 1403 trans-eGenes and 57729 unique SNPs.

For each protein target, I clustered univariate signals into 200 kb loci around the leading SNP. As a result, 1029 unique loci were found across all genes including 5077 loci-gene pairs. For each loci-gene pair, I found the minimum p value from PCO corresponding to the module containing the target protein. 4581 (90.2%) were replicated in PCO (p < 0.05/5077); 4239 (83.5%) were also significant in PCO (p < 5e-8/27), 3366/5077 (66.3%) has p value < univariate p. 3674/5077 with PCO p > 0 (some p = 0 due to precision issue).
```{r echo=F, fig.align='center', fig.cap="Signals found by univariate and PCO methods (02_pco_univar_comp.R)"}
knitr::include_graphics(c("assets/univar_pco_signal_comp.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Replication of univariate trans signals using PCO (02_pco_univar_comp.R)"}
knitr::include_graphics(c("assets/univar_pco_comp.png"), error = FALSE)
```

```{r echo=F, fig.align='center', fig.cap="Minimum p value comparison for 3674 loci (02_pco_univar_comp.R)"}
knitr::include_graphics(c("assets/minp_comp.png"), error = FALSE)
```

For each protein module in PCO analysis, a similar signal clustering was done to obtain 657 loci including 1595 loci-module pairs.

```{r echo=F, fig.align='center', fig.cap="Comparison of univariate and PCO P values (04_mahattan.R)"}
knitr::include_graphics(c("assets/manha_comp.png"), error = FALSE)
```

Then I investigated the number of significant proteins in each unique genome locus, and how this is related to PCO performance.
```{r echo=F, fig.align='center', fig.cap="Number of significant proteins per locus in univariate analysis for PCO significant and non-significant loci (02_pco_univar_comp.R)"}
knitr::include_graphics(c("assets/n_prot_per_loc.png"), error = FALSE)
```
Number of protein-loci pairs per locus for PCO significant loci: 

Mean = 7.3; 25% percentile = 1; median = 2; 75% percentile = 4; max = 984 (# of unique proteins = 571; chr17:26588663-26788663)

Number of significant proteins per locus for PCO non-significant loci: 

Mean = 1.4; 25% percentile = 1; median = 1; 75% percentile = 1; max = 9

### Heritability of protein
For 4782 protein/protein complex, there are 3816 with h2 > 0. Across all proteins, mean h2 = 0.098, median = 0.095, 25% quantile = 0.016, 75% quantile = 0.17, max = 0.81 (gene CTSB, a member of the C1 family of peptidases).
```{r echo=F, fig.align='center', fig.cap="Protein heritability (03_h2.R)"}
knitr::include_graphics(c("assets/h2_dist.png"), error = FALSE)
```
I tried two sets of annotations to estimate h2: 53 baseline annotations (Finucane et al., 2018) and 97 baseline annotations (Hujoel et al., 2019). They agree relatively well with each other (cor = 0.59).
```{r echo=F, fig.align='center', fig.cap="Cor of h2 between two annotations (03_h2.R)"}
knitr::include_graphics(c("assets/h2_cor_annot.png"), error = FALSE)
```

I calculated the h2 for two datasets (Sun et al., 2018; Gudjonsson et al., 2022). Sun's data has 3023 protein coding genes, 3301 individuals. They found 1927 significant associations, 34% of which are trans. Gudjonsson's data has 4427 protein coding genes, 5368 individuals. They found 4035 significant associations, 65% of which are trans.

Define cis as SNPs within the same chromosome as the gene and trans as SNPs in different chromosomes:

For Sun et al., 2018: mean cis h2 = 0.016; mean trans h2 = 0.091; mean trans h2 prop = 0.86

For Gudjonsson et al., 2018: mean cis h2 = 0.043; mean trans h2 = 0.42; mean trans h2 prop = 0.91

```{r echo=F, fig.align='center', fig.cap="h2 summary for two datasets (07_h2_2_datasets.R)"}
knitr::include_graphics(c("assets/h2_2_data.png"), error = FALSE)
```

Define cis as SNPs within 1 Mb of the gene tss and trans as SNPs outside 5 Mb of the gene tss and in different chromosomes:

For Sun et al., 2018: mean cis h2 = 0.011; mean trans h2 = 0.096; mean trans h2 prop = 0.90

For Gudjonsson et al., 2022: mean cis h2 = 0.017; mean trans h2 = 0.45; mean trans h2 prop = 0.96

For UKBiobank, 2022: mean cis h2 = 0.012; mean trans h2 = 0.091; mean trans h2 prop = 0.89

```{r echo=F, fig.align='center', fig.cap="h2 summary for 3 datasets (07_h2_3_datasets.R)"}
knitr::include_graphics(c("assets/h2_2_data_1mb_5mb.png"), error = FALSE)
```

